# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2012-07-30 09:18+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: interfaces/cli/get.py:75
msgid "Daemon connection lost, bailing out."
msgstr ""

#: interfaces/cli/get.py:81
msgid "Could not get daemon status"
msgstr ""

#: interfaces/cli/get.py:127
msgid "Could not obtain daemon status"
msgstr ""

#: interfaces/cli/get.py:192
msgid "LicornÂ® {0}, Python {1} on {2}"
msgstr ""

#: interfaces/cli/get.py:214
msgid "unable to write history file!"
msgstr ""

#: interfaces/cli/__init__.py:49 daemon/base.py:85
msgid "{0}: switched long_output status to {1}."
msgstr ""

#: interfaces/cli/__init__.py:51 core/_modules.py:675
#: extensions/openssh.py:123 daemon/base.py:87
msgid "enabled"
msgstr ""

#: interfaces/cli/__init__.py:53 interfaces/cli/argparser.py:978
#: core/_modules.py:675 extensions/openssh.py:125 daemon/base.py:89
msgid "disabled"
msgstr ""

#: interfaces/cli/__init__.py:54 daemon/base.py:90
msgid "toggle [dump status] long ouput on or off"
msgstr ""

#: interfaces/cli/__init__.py:67 daemon/base.py:119
msgid "{0}: increased verbosity level to {1}."
msgstr ""

#: interfaces/cli/__init__.py:74 daemon/base.py:126
msgid "{0}: verbosity level already at the maximum value ({1})."
msgstr ""

#: interfaces/cli/__init__.py:79 daemon/base.py:131
msgid "increase the daemon console verbosity level"
msgstr ""

#: interfaces/cli/__init__.py:94 daemon/base.py:138
msgid "{0}: decreased verbosity level to {1}."
msgstr ""

#: interfaces/cli/__init__.py:101 daemon/base.py:145
msgid "{0}: verbosity level already at the minimum value ({1})."
msgstr ""

#: interfaces/cli/__init__.py:106 daemon/base.py:150
msgid "decrease the daemon console verbosity level"
msgstr ""

#: interfaces/cli/__init__.py:112
msgid "end the interactive session cleanly"
msgstr ""

#: interfaces/cli/__init__.py:142
#, python-format
msgid "Unknow mode %s!"
msgstr ""

#: interfaces/cli/argparser.py:61
msgid "Filter options"
msgstr ""

#: interfaces/cli/argparser.py:69 interfaces/cli/argparser.py:77
#, python-format
msgid "Also select system data. I.e. output system %s too."
msgstr ""

#: interfaces/cli/argparser.py:83
msgid ""
"exclude {0} from the selection. Can be IDs or {1} names. Separated by commas "
"without spaces."
msgstr ""

#: interfaces/cli/argparser.py:91 interfaces/cli/argparser.py:98
#, python-format
msgid "long output (all info, attributes, etc). Default: %s."
msgstr ""

#: interfaces/cli/argparser.py:92 interfaces/cli/argparser.py:99
#: interfaces/cli/argparser.py:105 interfaces/cli/argparser.py:125
#: core/machines.py:1198
msgid "no"
msgstr ""

#: interfaces/cli/argparser.py:103
#, python-format
msgid ""
"Stay connected to the daemon and update the status every given interval (see "
"below). Default: %s."
msgstr ""

#: interfaces/cli/argparser.py:109
#, python-format
msgid "Monitoring interval (in seconds). Default: %s."
msgstr ""

#: interfaces/cli/argparser.py:110
msgid "1 sec"
msgstr ""

#: interfaces/cli/argparser.py:114
#, python-format
msgid "Monitor a given number of count. Default: %s."
msgstr ""

#: interfaces/cli/argparser.py:115 interfaces/cli/argparser.py:120
msgid "infinite"
msgstr ""

#: interfaces/cli/argparser.py:119
#, python-format
msgid "Monitor during a time period, in seconds. Default: %s."
msgstr ""

#: interfaces/cli/argparser.py:124
#, python-format
msgid "Do not clear the screen between each monitor output. Default: %s."
msgstr ""

#: interfaces/cli/argparser.py:131
msgid "{0} {1}: this can be a very long operation{2}"
msgstr ""

#: interfaces/cli/argparser.py:132
#, python-format
msgid "Check *all* %s on the system."
msgstr ""

#: interfaces/cli/argparser.py:135
msgid "Check every bit of the configuration."
msgstr ""

#: interfaces/cli/argparser.py:136
msgid "WARNING"
msgstr ""

#: interfaces/cli/argparser.py:137
#, python-format
msgid ", depending of the current number of %s."
msgstr ""

#: interfaces/cli/argparser.py:143
msgid ""
"Exclude {0} from the selection. Can be IDs or {1} names. Separated by commas "
"without spaces."
msgstr ""

#: interfaces/cli/argparser.py:151
msgid "Specify user(s) by their login (separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:156
msgid "Specify user(s) by their UID (separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:162
msgid "Only select system accounts."
msgstr ""

#: interfaces/cli/argparser.py:167
msgid "Only select non-system accounts."
msgstr ""

#: interfaces/cli/argparser.py:177
msgid ""
"Specify account(s) excluded from current operation, by their *login* "
"(separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:183
msgid ""
"Specify account(s) excluded from current operation by their UID (separated "
"by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:191
msgid "grep / fuzzy word match on the login/name/hostname."
msgstr ""

#: interfaces/cli/argparser.py:195
msgid "exclude login/name/hostname if grep / fuzzy word match."
msgstr ""

#: interfaces/cli/argparser.py:201 interfaces/cli/argparser.py:214
msgid "Include all watched {0}."
msgstr ""

#: interfaces/cli/argparser.py:205 interfaces/cli/argparser.py:218
msgid ""
"Include all non-watched {0}. Handy for checking all of them at once in a "
"cron script."
msgstr ""

#: interfaces/cli/argparser.py:225
msgid "Specify group(s) by their name (separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:230
msgid "Specify group(s) by their GID (separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:235
msgid "Only select system groups."
msgstr ""

#: interfaces/cli/argparser.py:240
msgid "Only select non-system groups."
msgstr ""

#: interfaces/cli/argparser.py:245
msgid "Only select privileged groups."
msgstr ""

#: interfaces/cli/argparser.py:252
msgid "Only select non-privileged groups."
msgstr ""

#: interfaces/cli/argparser.py:257
msgid "Only select responsibles groups."
msgstr ""

#: interfaces/cli/argparser.py:263
msgid "Only select non-responsible groups."
msgstr ""

#: interfaces/cli/argparser.py:267
msgid "Only select guests groups."
msgstr ""

#: interfaces/cli/argparser.py:272
msgid "Only select non-guest groups."
msgstr ""

#: interfaces/cli/argparser.py:276
msgid "Only select empty groups."
msgstr ""

#: interfaces/cli/argparser.py:285
msgid ""
"Specify group(s) to exclude from current operation, by their *name* "
"(separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:291
msgid ""
"Specify group(s) to exclude from current operation by their *GID* (separated "
"by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:298
#, python-format
msgid ""
"Specify profile by its common name (separated by commas without spaces, when "
"possible. If name contains spaces, use --group instead). %s."
msgstr ""

#: interfaces/cli/argparser.py:301
msgid "one of --name or --group is required"
msgstr ""

#: interfaces/cli/argparser.py:306
msgid ""
"specify profile by its primary group (separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:313
msgid ""
"Specify machine(s) by their hostname (separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:319
msgid ""
"Specify machine(s) by their IP address (separated by commas without spaces)."
msgstr ""

#: interfaces/cli/argparser.py:325
msgid "Only select machines whose status is totally unknown."
msgstr ""

#: interfaces/cli/argparser.py:329
msgid ""
"Only select offline machines (this includes shutdown,going to sleep, asleep, "
"shutting down and pyro shutdown machine)."
msgstr ""

#: interfaces/cli/argparser.py:336
msgid "Only select going to sleep machines."
msgstr ""

#: interfaces/cli/argparser.py:340
msgid "Only select asleep machines."
msgstr ""

#: interfaces/cli/argparser.py:344
#, python-format
msgid ""
"Only select online machines (%s, and includes booting, idle, active and "
"loaded machines)."
msgstr ""

#: interfaces/cli/argparser.py:346
msgid "this is the default behaviour"
msgstr ""

#: interfaces/cli/argparser.py:350
msgid "Only select booting machines."
msgstr ""

#: interfaces/cli/argparser.py:354
msgid "Only select idle machines."
msgstr ""

#: interfaces/cli/argparser.py:358
msgid "Only select active machines."
msgstr ""

#: interfaces/cli/argparser.py:362
msgid "Only select loaded machines."
msgstr ""

#: interfaces/cli/argparser.py:368
msgid "Only select 'extinction' tasks"
msgstr ""

#: interfaces/cli/argparser.py:381
msgid "options --force and --batch are mutually exclusive!"
msgstr ""

#: interfaces/cli/argparser.py:395
msgid "option --filename cannot be used with other individual specifiers."
msgstr ""

#: interfaces/cli/argparser.py:417
msgid ""
"(you can use partial and fuzzy matching on modes, eg. \"{0}\" will call "
"\"{1}\", \"{2}\" will match \"{3}\", and so on)"
msgstr ""

#: interfaces/cli/argparser.py:436
msgid "Output options"
msgstr ""

#: interfaces/cli/argparser.py:437
msgid "Modify how data is printed/exported."
msgstr ""

#: interfaces/cli/argparser.py:443
#, python-format
msgid ""
"Like previous option, but export the configuration subset in the shortest "
"form factor (only the values when possible ; %s)."
msgstr ""

#: interfaces/cli/argparser.py:446
msgid "this is the default"
msgstr ""

#: interfaces/cli/argparser.py:451
#, python-format
msgid ""
"When using configuration %s to output a subset of the system configuration, "
"export it in a useful way to be used in a bourne shell environment (i.e. "
"export VAR=\"value\")."
msgstr ""

#: interfaces/cli/argparser.py:454
msgid "sub-category"
msgstr ""

#: interfaces/cli/argparser.py:459
msgid ""
"Like previous option, but export the configuration subset in a useful way to "
"be used in a C shell environment (i.e. setenv VAR \"value\")."
msgstr ""

#: interfaces/cli/argparser.py:466
msgid ""
"Like previous option, but export the configuration subset in a usefull way "
"to be included in PHP code (i.e. $VAR=\"value\", use it with eval(`â¦`))."
msgstr ""

#: interfaces/cli/argparser.py:472
#, python-format
msgid ""
"Output data as XML (no colors, no verbose). Default: %s, output for CLI, "
"human-readable."
msgstr ""

#: interfaces/cli/argparser.py:474
msgid "NO XML"
msgstr ""

#: interfaces/cli/argparser.py:478
msgid ""
"Dump nearly RAW data on stdout. Used for debugging internal data structures. "
"WARNING: dump output can easily  flood your terminal."
msgstr ""

#: interfaces/cli/argparser.py:519
msgid "Events listing/monitoring options"
msgstr ""

#: interfaces/cli/argparser.py:524
msgid ""
"Stay connected to the daemon and dump events in \"monitor\" mode. You can "
"filter events with -f. Without this flag, all  valid events will only be "
"listed, with their handlers and callbacks if you add -v."
msgstr ""

#: interfaces/cli/argparser.py:531
#, python-format
msgid ""
"Specify facilities when monitor. Default: %s (see online documentation for "
"possible values)."
msgstr ""

#: interfaces/cli/argparser.py:533
msgid "std"
msgstr ""

#: interfaces/cli/argparser.py:748
msgid ""
"\n"
"\t{1}\n"
"\n"
"\t{0} user [--login] <login>\n"
"\n"
"\t{0} user [--login] <login>\n"
"\t\t[-s|--system]\n"
"\t\t[-p|--password \"<cleartext_password>\"]\n"
"\t\t[-g|--gid=<primary_gid|primary_group>]\n"
"\t\t[-r|--profile=<profile_name|profile_group|profile_gid>]\n"
"\t\t[-K|--skel=<absolute_path_to_skel>]\n"
"\t\t[-e|--gecos=<given name>]\n"
"\t\t[-H|--home=<home_dir>]\n"
"\t\t[other optionsâ¦]\n"
"\n"
"\t{2}\n"
"\t(Will automatically create users groups during the process)\n"
"\n"
"\t{0} users --filename=<file> --profile=<profile_name|profile_group|"
"profile_gid>\n"
"\t\t[--lastname-column=<COL>] [--firstname-column=<COL>]\n"
"\t\t[--group-column=<COL>] [--login-column=<COL>]\n"
"\t\t[--password-column=<COL>]\n"
"\t\t[--separator=<SEP>]\n"
"\t\t[--confirm-import]\n"
"\t\t[--no-sync]\n"
msgstr ""

#: interfaces/cli/argparser.py:774
msgid "Individual user creation:"
msgstr ""

#: interfaces/cli/argparser.py:775
msgid "Massive users creation:"
msgstr ""

#: interfaces/cli/argparser.py:784
msgid "Individual user creation options"
msgstr ""

#: interfaces/cli/argparser.py:788
#, python-format
msgid "Specify user's login (%s)."
msgstr ""

#: interfaces/cli/argparser.py:790
msgid "one of login or firstname+lastname arguments is required"
msgstr ""

#: interfaces/cli/argparser.py:794
#, python-format
msgid ""
"Specify user's GECOS field. If given, GECOS takes precedence on --firstname "
"and --lastname, which will be silently discarded. Default: autogenerated "
"from firstname & lastname if given, else %s."
msgstr ""

#: interfaces/cli/argparser.py:798
msgid "autogenerated from login"
msgstr ""

#: interfaces/cli/argparser.py:802
#, python-format
msgid "Specify user's password (else will be autogenerated, %d chars long)."
msgstr ""

#: interfaces/cli/argparser.py:808
#, python-format
msgid ""
"Specify user's future primary group (at your preference as a group name or a "
"GID). This parameter is overriden by the profile argument if you specify "
"both. Default: %s."
msgstr ""

#: interfaces/cli/argparser.py:816
msgid ""
"Specify future user's auxilliary groups (at your preference as groupnames or "
"GIDs, which can be mixed, and separated by commas without spaces). These "
"supplemental groups are added to the list of groups defined by the profile, "
"if you specify both. Default: None."
msgstr ""

#: interfaces/cli/argparser.py:824
msgid "Create a system account instead of a standard user (root only)."
msgstr ""

#: interfaces/cli/argparser.py:828
#, python-format
msgid ""
"Profile which will be applied to the user. Default: None. This argument "
"overrides the primary group / GID if both are specified. %s."
msgstr ""

#: interfaces/cli/argparser.py:831
msgid "It can be used for massive imports, too"
msgstr ""

#: interfaces/cli/argparser.py:835
msgid ""
"manually specify an UID for the new user. This UID must be free and inside "
"the range {0} - {1} for a standard user, and outside the range for a system "
"account, else it will be rejected and the user account won't be created. "
"Default: next free UID in the selected range."
msgstr ""

#: interfaces/cli/argparser.py:845
#, python-format
msgid ""
"Specify the user's home directory. Only valid for a system account, else "
"discarded because standard accounts have a fixed home dir \"%s\"."
msgstr ""

#: interfaces/cli/argparser.py:852
#, python-format
msgid ""
"Specify user's shell, from the ones given by command `get config shells`. "
"Default: %s"
msgstr ""

#: interfaces/cli/argparser.py:858
#, python-format
msgid ""
"Specify a particular skeleton to apply to home dir after creation, instead "
"of the profile or the primary-group implicit skel. Default: the profile skel "
"if a profile is given, else %s."
msgstr ""

#: interfaces/cli/argparser.py:866
msgid ""
"Specify user's first name (required if --lastname is given, overriden by "
"GECOS)."
msgstr ""

#: interfaces/cli/argparser.py:871
msgid ""
"Specify user's last name (required if --firstname is given, overriden by "
"GECOS)."
msgstr ""

#: interfaces/cli/argparser.py:876
msgid ""
"do not create the home directory when creating account. This is valid only "
"for system accounts, because standard accounts must always have a home."
msgstr ""

#: interfaces/cli/argparser.py:883
msgid ""
"Do not watch the home directory of the user for content changes. Usefull for "
"big directories which slow down licornd. You have to manually run {0} on non-"
"watched users. (default: {1})"
msgstr ""

#: interfaces/cli/argparser.py:888 interfaces/cli/argparser.py:1022
msgid "watched"
msgstr ""

#: interfaces/cli/argparser.py:892
#, python-format
msgid "The user will have no password. %s."
msgstr ""

#: interfaces/cli/argparser.py:893 interfaces/cli/argparser.py:900
msgid ""
"Currently this flag is ignored; it exists only for compatibility purposes "
"with old scripts"
msgstr ""

#: interfaces/cli/argparser.py:898
#, python-format
msgid "The user will not be able to login after the account creation. %s"
msgstr ""

#: interfaces/cli/argparser.py:911
msgid ""
"Specify backend in which to store the user account (default: {0}; possible "
"choices: {1}; {2})."
msgstr ""

#: interfaces/cli/argparser.py:916 interfaces/cli/argparser.py:1061
msgid "this argument can be used for massive imports, too"
msgstr ""

#: interfaces/cli/argparser.py:922
msgid "Massive users creation options"
msgstr ""

#: interfaces/cli/argparser.py:928
msgid "Required in massive imports mode"
msgstr ""

#: interfaces/cli/argparser.py:926
#, python-format
msgid ""
"path of the file you want to import accounts from (must point to a valid CSV "
"file). %s."
msgstr ""

#: interfaces/cli/argparser.py:932
#, python-format
msgid "lastname column number in CSV file (default is %s)."
msgstr ""

#: interfaces/cli/argparser.py:937
#, python-format
msgid "firstname column number in CSV (default is %s)."
msgstr ""

#: interfaces/cli/argparser.py:942
msgid "{0} column number in CSV file (default is {1})."
msgstr ""

#: interfaces/cli/argparser.py:948
msgid ""
"{0} column number in CSV file (default: {1}, login will be guessed from "
"firstname and lastname)."
msgstr ""

#: interfaces/cli/argparser.py:955
msgid ""
"{0} column number in CSV file (default: {1}: password will be randomly "
"generated and {2} chars long)."
msgstr ""

#: interfaces/cli/argparser.py:957 daemon/rwi.py:930 daemon/rwi.py:943
msgid "password"
msgstr ""

#: interfaces/cli/argparser.py:958
msgid "none"
msgstr ""

#: interfaces/cli/argparser.py:963
#, python-format
msgid "Separator for the CSV fields (default: %s, by sniffing in the file)."
msgstr ""

#: interfaces/cli/argparser.py:964
msgid "determined automatically"
msgstr ""

#: interfaces/cli/argparser.py:968
#, python-format
msgid ""
"Really do the import. %s on the system, but only give you an example of what "
"will be done, which is useful to verify your file has been correctly parsed "
"(fields order, separatorâ¦)."
msgstr ""

#: interfaces/cli/argparser.py:972
msgid "Without this flag the program will do nothing"
msgstr ""

#: interfaces/cli/argparser.py:976
#, python-format
msgid ""
"Commit changes only after all modifications (currently %s because extensions "
"need things to be commited to work)."
msgstr ""

#: interfaces/cli/argparser.py:1005
#, python-format
msgid "Specify group's name (%s)."
msgstr ""

#: interfaces/cli/argparser.py:1006 interfaces/cli/argparser.py:2070
msgid "required"
msgstr ""

#: interfaces/cli/argparser.py:1012 core/groups.py:2341
msgid "not permissive"
msgstr ""

#: interfaces/cli/argparser.py:1010
msgid "Will the shared group directory be permissive? (default: {0})."
msgstr ""

#: interfaces/cli/argparser.py:1017
msgid ""
"Do not watch the shared directory of the group for content changes. Usefull "
"for big directories which slow down licornd. You have to manually run {0} on "
"non-watched groups. (default: {1})"
msgstr ""

#: interfaces/cli/argparser.py:1026
msgid "Specify the GID (root / @admin members only)."
msgstr ""

#: interfaces/cli/argparser.py:1030
#, python-format
msgid "Description of the group (free text, but must match /%s/i)."
msgstr ""

#: interfaces/cli/argparser.py:1037
#, python-format
msgid "skeleton directory for the group (default: %s)."
msgstr ""

#: interfaces/cli/argparser.py:1042
#, python-format
msgid ""
"The group will be a system group (only root or %s members can decide this, "
"but this is not enforced yet)."
msgstr ""

#: interfaces/cli/argparser.py:1048
msgid "Users to make members of this group just after creation."
msgstr ""

#: interfaces/cli/argparser.py:1056
msgid ""
"Specify backend in which to store the group (default: {0}; possible choices: "
"{1}; {2})."
msgstr ""

#: interfaces/cli/argparser.py:1082
msgid "Add profile options"
msgstr ""

#: interfaces/cli/argparser.py:1086
msgid ""
"The profile's name (ie: Â«Â Administrator Â», Â«Â Power userÂ Â», Â«Â WebmasterÂ Â», "
"Â«Â GuestÂ Â»). It must conform to {0} word. {1}."
msgstr ""

#: interfaces/cli/argparser.py:1089 interfaces/cli/argparser.py:1096
#: interfaces/cli/argparser.py:1260
msgid "It is required"
msgstr ""

#: interfaces/cli/argparser.py:1093
#, python-format
msgid ""
"Group name (or GID) identifying the profile on the system (ie: "
"Â«administratorsÂ», Â«power-usersÂ», Â«webmastersÂ», Â«guestsÂ»). It should be a "
"plural world and will become a system group. %s."
msgstr ""

#: interfaces/cli/argparser.py:1100
#, python-format
msgid "Description of the profile (free text, but must conform to /%s/i)."
msgstr ""

#: interfaces/cli/argparser.py:1107
#, python-format
msgid "Default shell for this profile (defaults: %s)."
msgstr ""

#: interfaces/cli/argparser.py:1112
#, python-format
msgid "User data quota in Mb (a soft quota, defaults: %s)."
msgstr ""

#: interfaces/cli/argparser.py:1117
msgid ""
"Groups users of this profile will become members of.Separated by commas "
"without spaces."
msgstr ""

#: interfaces/cli/argparser.py:1123
#, python-format
msgid ""
"Skeleton dir for this profile (must be an absolute path, defaults to %s)."
msgstr ""

#: interfaces/cli/argparser.py:1129
#, python-format
msgid ""
"Confirm the use of a previously created system group for the profile. %s, "
"but in some cases it is needed and perfectly safe (when the group is created "
"by another package or script)."
msgstr ""

#: interfaces/cli/argparser.py:1133
msgid "This can be risky in some situations"
msgstr ""

#: interfaces/cli/argparser.py:1149 interfaces/cli/argparser.py:1183
msgid "Add keyword options"
msgstr ""

#: interfaces/cli/argparser.py:1153
#, python-format
msgid "The keyword's name. It should be a singular word and %s."
msgstr ""

#: interfaces/cli/argparser.py:1154
msgid "it is required"
msgstr ""

#: interfaces/cli/argparser.py:1158
msgid "Keyword's parent name."
msgstr ""

#: interfaces/cli/argparser.py:1162
msgid "Description of the keyword (free text)."
msgstr ""

#: interfaces/cli/argparser.py:1170
msgid ""
"\n"
"\t{0} backup [--force] [{1}]\n"
"\n"
"\tWhere {1} is optionnal, and will bu fuzzy matched against device names, "
"volume labels and GUIDs if provided. If not specified, the program will auto-"
"select the first available volume."
msgstr ""

#: interfaces/cli/argparser.py:1187
msgid ""
"The event name. A singular word with underscores but no spaces or commas. "
"{0}. {1} will display a list of valid event names."
msgstr ""

#: interfaces/cli/argparser.py:1190
msgid "Required"
msgstr ""

#: interfaces/cli/argparser.py:1191
msgid "get events"
msgstr ""

#: interfaces/cli/argparser.py:1196
msgid ""
"The event unnamed arguments (usually known as 'args' in Python).\n"
"\n"
"It should be a string representing a python list in JSON syntax (it will be "
"parsed with `json.loads()`).\n"
"\n"
"It's currently impossible to give resolvable  arguments (eg. `LMC.users."
"by_name('my_user')`) so you are stuck with only simple events to send. Some "
"of them support IDs and do dynamic resolution when found, but that's "
"currently not the global rule."
msgstr ""

#: interfaces/cli/argparser.py:1208
msgid ""
"The event keywords arguments, usually known as 'kwargs' in Python.\n"
"\n"
"The same JSON rules and restrictions apply, except that kwargs must "
"obviously represent a dictionnary."
msgstr ""

#: interfaces/cli/argparser.py:1214
msgid "The event priority, a string like 'LOW', 'NORMAL' (default), or 'HIGH'."
msgstr ""

#: interfaces/cli/argparser.py:1219
msgid ""
"Run the event synchronously, eg. wait for handlers and callbacks to "
"terminate before giving back hand. Default: {0}."
msgstr ""

#: interfaces/cli/argparser.py:1221
msgid "False"
msgstr ""

#: interfaces/cli/argparser.py:1229
msgid ""
"\n"
"\t{0} event [--name] <event_name> [optionsâ¦]\n"
"\n"
"Manually send en event inside the LicornÂ® daemon.\n"
"Use with caution, as you can generate any kind of event."
msgstr ""

#: interfaces/cli/argparser.py:1253
msgid "Add privilege options"
msgstr ""

#: interfaces/cli/argparser.py:1257
#, python-format
msgid ""
"The privilege's name(s), which is really a system group name (only system "
"groups can be promoted to privileges). %s and can be a single word or "
"multiple ones, separated by commas."
msgstr ""

#: interfaces/cli/argparser.py:1288
msgid ""
"Scan a network for online hosts and attach them to the system. Syntax: "
"192.168.0.0/24 or 10.0.0.0/8 and the like."
msgstr ""

#: interfaces/cli/argparser.py:1293
msgid "Scan the local area network(s), looking for unattached hosts."
msgstr ""

#: interfaces/cli/argparser.py:1319
msgid "Add volume(s) options"
msgstr ""

#: interfaces/cli/argparser.py:1344
msgid "Add task(s) options"
msgstr ""

#: interfaces/cli/argparser.py:1350 interfaces/cli/argparser.py:1353
#: interfaces/cli/argparser.py:1562
msgid "This argument is required"
msgstr ""

#: interfaces/cli/argparser.py:1349
msgid "Specify name for the task, {0}"
msgstr ""

#: interfaces/cli/argparser.py:1352
msgid "Specify action for the task, {0}"
msgstr ""

#: interfaces/cli/argparser.py:1356
msgid ""
"Specify year for the action, use the following syntax [*|*/n|n-m|n[,m[...]]]^"
"[n[,m[...]]]. Only used in mode 'single'"
msgstr ""

#: interfaces/cli/argparser.py:1359
msgid ""
"Specify month for the action, use the following syntax [*|*/n|n-m|n[,m[...]]]"
"^[n[,m[...]]].. Only used in mode 'single'"
msgstr ""

#: interfaces/cli/argparser.py:1362
msgid ""
"Specify day for the action, use the following syntax [*|*/n|n-m|n[,m[...]]]^"
"[n[,m[...]]].. Only used in mode 'single'"
msgstr ""

#: interfaces/cli/argparser.py:1365
msgid ""
"Specify hour for the action, use the following syntax [*|*/n|n-m|n[,m[...]]]^"
"[n[,m[...]]].."
msgstr ""

#: interfaces/cli/argparser.py:1367
msgid ""
"Specify minute for the action, use the following syntax [*|*/n|n-m|n[,m"
"[...]]]^[n[,m[...]]].."
msgstr ""

#: interfaces/cli/argparser.py:1369
msgid ""
"Specify second for the action, use the following syntax [*|*/n|n-m|n[,m"
"[...]]]^[n[,m[...]]].."
msgstr ""

#: interfaces/cli/argparser.py:1371
msgid ""
"Specify week days for the action, use the following syntax [*|*/n|n-m|n[,m"
"[...]]]^[n[,m[...]]].."
msgstr ""

#: interfaces/cli/argparser.py:1374
msgid "Specify a year to start the task, integer."
msgstr ""

#: interfaces/cli/argparser.py:1376
msgid "Specify a month to start the task, integer."
msgstr ""

#: interfaces/cli/argparser.py:1378
msgid "Specify a day to start the task, integer."
msgstr ""

#: interfaces/cli/argparser.py:1380 interfaces/cli/argparser.py:1382
msgid "Specify an hour to start the task, integer."
msgstr ""

#: interfaces/cli/argparser.py:1384
msgid "Specify a second to start the task, integer."
msgstr ""

#: interfaces/cli/argparser.py:1387
msgid "Specify args for the action, format: --args=\"arg1[;arg2[;arg3[...]]]\""
msgstr ""

#: interfaces/cli/argparser.py:1389
msgid ""
"Specify kwargs for the action, format: --kwargs=\"k1=v1[;k2=v2[;k3=v3"
"[...]]]\""
msgstr ""

#: interfaces/cli/argparser.py:1391
msgid ""
"Specify if arguments  will be resolved during execution or during task's "
"load. Default is during task's execution"
msgstr ""

#: interfaces/cli/argparser.py:1405
msgid "You must specify one of \"--name\" or \"--action\"!"
msgstr ""

#: interfaces/cli/argparser.py:1423 interfaces/cli/argparser.py:1445
msgid "Delete user options"
msgstr ""

#: interfaces/cli/argparser.py:1427
#, python-format
msgid ""
"Don't make a backup of user's home directory in %s (Default: home directory "
"will be archived)."
msgstr ""

#: interfaces/cli/argparser.py:1448
msgid "name of the task to delete)."
msgstr ""

#: interfaces/cli/argparser.py:1451
msgid "id of the task to delete)."
msgstr ""

#: interfaces/cli/argparser.py:1467
msgid "Delete group options"
msgstr ""

#: interfaces/cli/argparser.py:1471
msgid ""
"get the list of groups from a file (can be stdin, just specify '-' as the "
"filename)."
msgstr ""

#: interfaces/cli/argparser.py:1476
msgid ""
"Delete the group primary members (user accounts) too (default: {0}, the "
"program will annoy you with a sane warning."
msgstr ""

#: interfaces/cli/argparser.py:1478
msgid "NO"
msgstr ""

#: interfaces/cli/argparser.py:1482
msgid ""
"Don't make a backup of users home directories in {0} when deleting members "
"(default: {1})."
msgstr ""

#: interfaces/cli/argparser.py:1486
msgid "archive deleted group shared directories"
msgstr ""

#: interfaces/cli/argparser.py:1505
msgid "Delete profile options"
msgstr ""

#: interfaces/cli/argparser.py:1509
#, python-format
msgid "the profile's users will be deleted (%s)."
msgstr ""

#: interfaces/cli/argparser.py:1510
msgid "Use with caution"
msgstr ""

#: interfaces/cli/argparser.py:1514
msgid "Don't make a backup of user's home when deleting them."
msgstr ""

#: interfaces/cli/argparser.py:1530
msgid "Delete keyword options"
msgstr ""

#: interfaces/cli/argparser.py:1534
msgid "Specify the keyword to delete."
msgstr ""

#: interfaces/cli/argparser.py:1538
msgid "Delete the parent and his children."
msgstr ""

#: interfaces/cli/argparser.py:1556
msgid "Delete privilege options"
msgstr ""

#: interfaces/cli/argparser.py:1560
#, python-format
msgid ""
"The privilege's name(s). %s and can be a single word or multiple ones, "
"separated by commas."
msgstr ""

#: interfaces/cli/argparser.py:1599
msgid "Modify user options"
msgstr ""

#: interfaces/cli/argparser.py:1603
#, python-format
msgid ""
"Specify user's new password on the command line (%s, it can be written in "
"your shell history file)."
msgstr ""

#: interfaces/cli/argparser.py:1609
msgid ""
"Ask for a new password for the user. If changing your own password, you will "
"be asked for the old, too."
msgstr ""

#: interfaces/cli/argparser.py:1614
msgid "let the system generate a random password for this user."
msgstr ""

#: interfaces/cli/argparser.py:1619
#, python-format
msgid "choose the new password length (default %s)."
msgstr ""

#: interfaces/cli/argparser.py:1624
msgid "Specify user's new GECOS string (generaly first and last names)."
msgstr ""

#: interfaces/cli/argparser.py:1629
#, python-format
msgid ""
"Specify user's new shell (generaly /bin/something, must be taken from %s)."
msgstr ""

#: interfaces/cli/argparser.py:1635
msgid ""
"lock the account (user wn't be able to login under Linux and Windows/MAC "
"until unlocked)."
msgstr ""

#: interfaces/cli/argparser.py:1640
msgid "unlock the user account and restore login ability."
msgstr ""

#: interfaces/cli/argparser.py:1645
msgid "(Re-)watch the home directory of the user for content changes."
msgstr ""

#: interfaces/cli/argparser.py:1650
msgid ""
"Do not watch the home directory of the user for content changes. Usefull for "
"big directory which slow down licornd. You have to manually run {0} on non-"
"watched groups."
msgstr ""

#: interfaces/cli/argparser.py:1657
msgid "make user member of these groups."
msgstr ""

#: interfaces/cli/argparser.py:1661
msgid "remove user from these groups."
msgstr ""

#: interfaces/cli/argparser.py:1665
msgid ""
"Re-apply the user's skel, or another skel you specify on the command line. "
"(use with caution, it will overwrite the dirs/files provided by the skel in "
"the user's home dir."
msgstr ""

#: interfaces/cli/argparser.py:1672
msgid ""
"Restore the INotifier watch for this account home directory. This is "
"particularly useful after a directory move."
msgstr ""

#: interfaces/cli/argparser.py:1699
msgid "Modify machine(s) options"
msgstr ""

#: interfaces/cli/argparser.py:1703
msgid "remotely shutdown specified machine(s)."
msgstr ""

#: interfaces/cli/argparser.py:1707
msgid ""
"Display a warning message to connected user(s) before shutting system(s) "
"down."
msgstr ""

#: interfaces/cli/argparser.py:1715
msgid "remotely upgrade pending packages (do security updates)."
msgstr ""

#: interfaces/cli/argparser.py:1746
msgid "Modify volume(s) options"
msgstr ""

#: interfaces/cli/argparser.py:1750
#, python-format
msgid ""
"Specify one or more volume(s) to enable (mark as available and reserved for "
"%s internal use), either by giving its device path or mount point."
msgstr ""

#: interfaces/cli/argparser.py:1757
#, python-format
msgid ""
"specify one or more volume(s) to disable (unmark as available for %s), "
"either by giving its device path or mount point."
msgstr ""

#: interfaces/cli/argparser.py:1764
msgid "specify one or more volume(s) to (re-)mount."
msgstr ""

#: interfaces/cli/argparser.py:1769
msgid "specify one or more volume(s) to unmount."
msgstr ""

#: interfaces/cli/argparser.py:1801
msgid "Modify group options"
msgstr ""

#: interfaces/cli/argparser.py:1805
msgid "Specify group's new name (not yet implemented)."
msgstr ""

#: interfaces/cli/argparser.py:1809
msgid "Specify group's new skel dir."
msgstr ""

#: interfaces/cli/argparser.py:1813
msgid "Specify new group's description"
msgstr ""

#: interfaces/cli/argparser.py:1817
msgid "Set the shared directory of the group permissive."
msgstr ""

#: interfaces/cli/argparser.py:1821
msgid "Set the shared directory of the group not permissive."
msgstr ""

#: interfaces/cli/argparser.py:1826
msgid "(Re-)watch the shared directory of the group for content changes."
msgstr ""

#: interfaces/cli/argparser.py:1831
msgid ""
"Do not watch the shared directory of the group for content changes. Usefull "
"for big directory which slow down licornd. You have to manually run {0} on "
"non-watched groups."
msgstr ""

#: interfaces/cli/argparser.py:1838
msgid ""
"Restore the INotifier watch for the group home directory. This is "
"particularly useful after a directory move."
msgstr ""

#: interfaces/cli/argparser.py:1843
msgid ""
"Add users to the group. The users are separated by commas without spaces."
msgstr ""

#: interfaces/cli/argparser.py:1848
msgid ""
"Delete users from the group. The users are separated by commas without "
"spaces."
msgstr ""

#: interfaces/cli/argparser.py:1853
msgid ""
"Add responsibles to the group. The responsibles are separated by commas "
"without spaces."
msgstr ""

#: interfaces/cli/argparser.py:1858
msgid ""
"Delete responsibles from the group. The responsibles are separated by commas "
"without spaces."
msgstr ""

#: interfaces/cli/argparser.py:1863
msgid ""
"Add guests to the group. The guests are separated by commas without spaces."
msgstr ""

#: interfaces/cli/argparser.py:1868
msgid ""
"Delete guests from the group. The guests are separated by commas without "
"spaces."
msgstr ""

#: interfaces/cli/argparser.py:1873
msgid ""
"Add the profiles which the users can access to the group's shared directory. "
"The profiles are separated by commas without spaces."
msgstr ""

#: interfaces/cli/argparser.py:1879
msgid ""
"Delete the profiles which the users can access to the group's shared "
"directory. The profiles are separated by commas without spaces."
msgstr ""

#: interfaces/cli/argparser.py:1888
#, python-format
msgid ""
"Move the group from its current backend to another, where it will definitely "
"be stored (specify new backend name as argument, taken from %s)."
msgstr ""

#: interfaces/cli/argparser.py:1917
msgid "specify profile's new name"
msgstr ""

#: interfaces/cli/argparser.py:1921
msgid "Rename primary group (NOT IMPLEMENTED)."
msgstr ""

#: interfaces/cli/argparser.py:1925
msgid "Change profile's description."
msgstr ""

#: interfaces/cli/argparser.py:1929
#, python-format
msgid ""
"Change profile shell (defaults to %s if you specify --shell without argument)"
"Instant-applyed to current members if not specified otherwise."
msgstr ""

#: interfaces/cli/argparser.py:1936
#, python-format
msgid ""
"Change profile's user quota (in Mb, defaults to %s if you specify --quota "
"without argument).Instant-applyed to current members if not specified "
"otherwise."
msgstr ""

#: interfaces/cli/argparser.py:1943
#, python-format
msgid ""
"Change profile skel (specify a skel dir as an absolute pathname, defaults to "
"%s if you give --skel without argument).Instant-applyed to current members "
"if not specified otherwise."
msgstr ""

#: interfaces/cli/argparser.py:1950
msgid ""
"Add one or more group(s) to default memberships of profile (separate groups "
"with commas without spaces).Instant-applyed to current members if not "
"specified otherwise."
msgstr ""

#: interfaces/cli/argparser.py:1956
msgid ""
"Delete one or more group(s) from default memberships of profile (separate "
"groups with commas without spaces). Instant-applyed to current members if "
"not specified otherwise."
msgstr ""

#: interfaces/cli/argparser.py:1962
msgid "Re-apply only the default group memberships of the profile."
msgstr ""

#: interfaces/cli/argparser.py:1966
msgid "Re-apply only the skel of the profile."
msgstr ""

#: interfaces/cli/argparser.py:1970
msgid "Re-apply all the profile's attributes (groups and skel)."
msgstr ""

#: interfaces/cli/argparser.py:1974
msgid ""
"Re-apply to specific users accounts (separate them with commas without "
"spaces)."
msgstr ""

#: interfaces/cli/argparser.py:1979
msgid ""
"Re-apply to all members of one or more groups (separate groups with commas "
"without spaces). You can mix --to-users and --to-groups."
msgstr ""

#: interfaces/cli/argparser.py:1985
msgid "Re-apply to all users members of the profile."
msgstr ""

#: interfaces/cli/argparser.py:1989
msgid "Re-apply to all user accounts on the system (LENGHTY operation !)."
msgstr ""

#: interfaces/cli/argparser.py:1994
#, python-format
msgid ""
"Don't apply group addition/deletion and other parameters instantly to all "
"members of the modified profile (%s; use this only if you know what you're "
"doing)."
msgstr ""

#: interfaces/cli/argparser.py:2001
msgid "Commit changes only after all modifications (currently disabled)."
msgstr ""

#: interfaces/cli/argparser.py:2024 interfaces/cli/argparser.py:2065
msgid "Modify keyword options"
msgstr ""

#: interfaces/cli/argparser.py:2028
#, python-format
msgid "Specify keyword to modify (%s)."
msgstr ""

#: interfaces/cli/argparser.py:2033
msgid "Rename keyword"
msgstr ""

#: interfaces/cli/argparser.py:2037
msgid "Change keyword's parent."
msgstr ""

#: interfaces/cli/argparser.py:2041 interfaces/cli/argparser.py:2049
#: interfaces/cli/argparser.py:2090
msgid "Remove parent."
msgstr ""

#: interfaces/cli/argparser.py:2045
msgid "Modify all file in all subdirs."
msgstr ""

#: interfaces/cli/argparser.py:2069
#, python-format
msgid "specify path of the file/directory to tag (%s)."
msgstr ""

#: interfaces/cli/argparser.py:2074
msgid "Add keywords."
msgstr ""

#: interfaces/cli/argparser.py:2078
msgid "Remove keywords."
msgstr ""

#: interfaces/cli/argparser.py:2082
msgid "Remove all keywords."
msgstr ""

#: interfaces/cli/argparser.py:2086
msgid "Set modifications to all subdirs."
msgstr ""

#: interfaces/cli/argparser.py:2107 interfaces/cli/argparser.py:2240
msgid "Modify configuration options"
msgstr ""

#: interfaces/cli/argparser.py:2111
msgid ""
"create system groups, directories and settings from system configuration "
"(PARTIALLY OBSOLETED, TOTALLY UNSUPPORTED AND VERY DANGEROUS, USE WITH "
"CAUTION)."
msgstr ""

#: interfaces/cli/argparser.py:2117
msgid ""
"When creating system groups and directories, apply restrictive perms (710) "
"on shared dirs instead of relaxed ones (750)."
msgstr ""

#: interfaces/cli/argparser.py:2123
#, python-format
msgid "Set restrictive perms (710) on %s."
msgstr ""

#: interfaces/cli/argparser.py:2129
#, python-format
msgid "Set relaxed perms (750) on %s."
msgstr ""

#: interfaces/cli/argparser.py:2134
#, python-format
msgid ""
"Enable given backend(s) on the current system (separated by commas without "
"spaces). List of available backends with `%s`."
msgstr ""

#: interfaces/cli/argparser.py:2140
#, python-format
msgid ""
"Disable given backend(s) on the current system (separated by commas without "
"spaces). List of available backends with `%s`."
msgstr ""

#: interfaces/cli/argparser.py:2146
#, python-format
msgid ""
"Enable given extension(s) on the current system (separated by commas without "
"spaces). List of available extensions with `%s`."
msgstr ""

#: interfaces/cli/argparser.py:2152
#, python-format
msgid ""
"Disable given extension(s) on the current system (separated by commas "
"without spaces). List of available extensions with `%s`."
msgstr ""

#: interfaces/cli/argparser.py:2158
msgid "change machine hostname (not yet implemented)."
msgstr ""

#: interfaces/cli/argparser.py:2162
msgid "change machine's IP (for eth0 only) (not yet implemented)."
msgstr ""

#: interfaces/cli/argparser.py:2166
msgid "add privileges (system groups) to privileges whitelist."
msgstr ""

#: interfaces/cli/argparser.py:2171
msgid "remove privileges (system groups) from privileges whitelist."
msgstr ""

#: interfaces/cli/argparser.py:2244
msgid ""
"Launch Django tests against the WMI2 apps, separated by commas without "
"spaces. Just type \"all\" if you want it."
msgstr ""

#: interfaces/__init__.py:109
msgid "Harmless exception while releasing the LMC."
msgstr ""

#: interfaces/__init__.py:120
msgid "{0}: {1} received, restarting listener."
msgstr ""

#: interfaces/__init__.py:121
msgid "SIGUSR2"
msgstr ""

#: interfaces/__init__.py:226
msgid "Interrupted, cleaning up!"
msgstr ""

#: interfaces/gui/hkew.glade:7
msgid "Licorn Keywords File Modifier"
msgstr ""

#: interfaces/gui/hkew.glade:18
msgid "<span size=\"xx-large\">Licorn Keywords modifier</span>"
msgstr ""

#: interfaces/gui/hkew.glade:30
msgid "  Paths affected by keywords  "
msgstr ""

#: interfaces/gui/hkew.glade:59
msgid "Apply keywords recusively to dirs"
msgstr ""

#: interfaces/gui/hkew.glade:97
msgid "  Keywords (instant apply)  "
msgstr ""

#: interfaces/gui/hkew.glade:112 interfaces/gui/hkew.glade:222
msgid "Current selection:"
msgstr ""

#: interfaces/gui/hkew.glade:122 interfaces/gui/hkew.glade:232
msgid "<b>-</b>"
msgstr ""

#: interfaces/gui/hkew.glade:196
msgid "Licorn Keywords File Query"
msgstr ""

#: interfaces/gui/hkew.glade:207
msgid "<span size=\"xx-large\">Licorn Keywords File Query</span>"
msgstr ""

#: contrib/apt_check.py:24
msgid "Unknown Error: '{0}' ({1})"
msgstr ""

#: contrib/apt_check.py:95
msgid "Error: BrokenCount > 0"
msgstr ""

#: core/profiles.py:138
msgid "profile {0}: group {1} does not exist, ignored."
msgstr ""

#: core/profiles.py:198
msgid "A profile cannot have an empty name!"
msgstr ""

#: core/profiles.py:202 core/profiles.py:808
msgid "Malformed profile name \"{0}\", must match /{1}/i."
msgstr ""

#: core/profiles.py:218
msgid "Changed profile {0} name to {1}."
msgstr ""

#: core/profiles.py:238
msgid "You must specify a description (an empty text is OK, but not `None`)."
msgstr ""

#: core/profiles.py:242 core/profiles.py:818
msgid "Malformed profile description \"{0}\", must match /{1}/i."
msgstr ""

#: core/profiles.py:250
msgid "Changed profile {0} description to \"{1}\"."
msgstr ""

#: core/profiles.py:326
msgid ""
"You must specify a valid skel dir for the profile (use --help to know how)."
msgstr ""

#: core/profiles.py:330
msgid " Invalid skeleton \"{0}\". Valid skels are {1}."
msgstr ""

#: core/profiles.py:338
msgid "Changed profile {0} skeleton to {1}."
msgstr ""

#: core/profiles.py:342
#, python-format
msgid "Applying new skel to current members %s in the background."
msgstr ""

#: core/profiles.py:357
msgid ""
"You must specify a valid shell for the profile (use --help to know how)."
msgstr ""

#: core/profiles.py:361
msgid "Invalid shell \"{0}\". Allowed shells are {1}."
msgstr ""

#: core/profiles.py:369
msgid "Changed profile {0} shell to {1}."
msgstr ""

#: core/profiles.py:386
msgid "You must specify a quota as integer, not \"{}\"."
msgstr ""

#: core/profiles.py:393
msgid "Changed profile {0} quota to {1}Mb."
msgstr ""

#: core/profiles.py:407
msgid "You must specify a list of groups to add to the profile."
msgstr ""

#: core/profiles.py:416
msgid "Skipped group {0}, already in profile {1}."
msgstr ""

#: core/profiles.py:421
msgid "Cannot add group {0} to its own profile {1}."
msgstr ""

#: core/profiles.py:430
msgid "Added group {0} to profile {1}. {2}"
msgstr ""

#: core/profiles.py:434 core/profiles.py:471
msgid " Applying to current members in the background."
msgstr ""

#: core/profiles.py:452
msgid "You must specify a list of groups to delete from the profile."
msgstr ""

#: core/profiles.py:466
msgid "Deleted group {0} from profile {1}. {2}"
msgstr ""

#: core/profiles.py:479
msgid "Skipped group {0}, not in groups of profile {1}."
msgstr ""

#: core/profiles.py:494
msgid ""
"Not applying skel {0} of {1} profile on account {2} because this account is "
"not in this profile."
msgstr ""

#: core/profiles.py:504
msgid "Applying skel {0} to user {1}."
msgstr ""

#: core/profiles.py:516
#, python-format
msgid "Installing skel part %s."
msgstr ""

#: core/profiles.py:579
msgid ""
"Applyed skel {0} to user {1}, permissions are checked in the background."
msgstr ""

#: core/profiles.py:584
msgid "Skel {0} already applied or skipped for user {1}."
msgstr ""

#: core/profiles.py:593
msgid "{name} ({group}, gid={gid}): {descr}"
msgstr ""

#: core/profiles.py:598
msgid ""
"\t{home_label} {home}, {skel_label} {skel}, {shell_label} {shell}, "
"{quota_label} {quota}{quota_unit}{groups}"
msgstr ""

#: core/profiles.py:601
msgid "Home:"
msgstr ""

#: core/profiles.py:603
msgid "skeleton:"
msgstr ""

#: core/profiles.py:605
msgid "shell:"
msgstr ""

#: core/profiles.py:607
msgid "quota:"
msgstr ""

#: core/profiles.py:609
msgid "Mb"
msgstr ""

#: core/profiles.py:611
msgid "Groups:"
msgstr ""

#: core/profiles.py:661
msgid "profile"
msgstr ""

#: core/profiles.py:664
msgid "PrID"
msgstr ""

#: core/profiles.py:730
msgid "Adding a default {0} profile on the system (this is mandatory)."
msgstr ""

#: core/profiles.py:732 core/profiles.py:738
msgid "Users"
msgstr ""

#: core/profiles.py:741
msgid "Standard desktop users"
msgstr ""

#: core/profiles.py:792
#, python-format
msgid "The %s profile"
msgstr ""

#: core/profiles.py:798 core/users.py:419
msgid "Invalid shell \"{0}\". Valid shells are {1}."
msgstr ""

#: core/profiles.py:803
msgid "Invalid skeleton \"{0}\". Valid skels are {1}."
msgstr ""

#: core/profiles.py:813
msgid "Malformed profile group \"{0}\", must match /{1}/i."
msgstr ""

#: core/profiles.py:830
msgid "Clamped negative quota value to absolute value {0}."
msgstr ""

#: core/profiles.py:836
msgid "Clamped quota to maximum value {0}Mb."
msgstr ""

#: core/profiles.py:840
msgid ""
"Malformed profile quota \"{0}\": must be an integer, specifying a size in Mb."
msgstr ""

#: core/profiles.py:863
#, python-format
msgid ""
"A profile with name \"%s\" already exists on the system. Please choose "
"another one."
msgstr ""

#: core/profiles.py:869
msgid ""
"The group {0} is already taken by profile {1}. Please choose another one."
msgstr ""

#: core/profiles.py:882
#, python-format
msgid ""
"A system group named \"%s\" already exists. Please choose another group name "
"for your profile, or use --force-existing to override and use it anyway "
"(this is not recommended because the group will be deleted when you delete "
"the profile)."
msgstr ""

#: core/profiles.py:902
msgid ""
"The group {0} (gid={1}) is not a system group. It cannot be used as primary "
"group for a profile. Please choose another one, or do not specify any, and a "
"default one will be automatically created."
msgstr ""

#: core/profiles.py:922
msgid "Added profile {0} (group={1}, gid={2})."
msgstr ""

#: core/profiles.py:945
#, python-format
msgid "Kept restricted system group %s."
msgstr ""

#: core/profiles.py:972
#, python-format
msgid "Deleted profile %s."
msgstr ""

#: core/profiles.py:993
msgid "You must choose to apply the groups or/and the skel"
msgstr ""

#: core/profiles.py:998
#, python-format
msgid ""
"Skipped system account %s (we do not reapply skels on them for safety "
"reasons)."
msgstr ""

#: core/profiles.py:1004
#, python-format
msgid "Skipped account %s, which has no profile."
msgstr ""

#: core/profiles.py:1071
msgid "You must specify a groupname or a profile name to test existence of."
msgstr ""

#: core/profiles.py:1079
#, python-format
msgid "Profile group %s does not exist."
msgstr ""

#: core/profiles.py:1087
#, python-format
msgid "Profile %s does not exist."
msgstr ""

#: core/profiles.py:1118
msgid "The tag <{0}> was not found in {1}."
msgstr ""

#: core/profiles.py:1130
msgid "The tag <{0}> must not have an empty value (in {1})."
msgstr ""

#: core/users.py:108
msgid ""
"Cannot build a valid login (got \"{0}\", which does not verify {1}) with the "
"firstname/lastname you provided (\"{2}\" and \"{3}\")."
msgstr ""

#: core/users.py:358
msgid ""
"{0}: tried to set {1} as {2} primary group, whose GID is not the same ({3} "
"vs. {4})."
msgstr ""

#: core/users.py:388
msgid "You must specify a gecos"
msgstr ""

#: core/users.py:392 core/users.py:1485
msgid "Malformed GECOS field \"{0}\", must match /{1}/i."
msgstr ""

#: core/users.py:403
msgid "Changed user {0} gecos to \"{1}\"."
msgstr ""

#: core/users.py:416
msgid "You must specify a shell"
msgstr ""

#: core/users.py:430
msgid "Changed user {0} shell to {1}."
msgstr ""

#: core/users.py:464
msgid ""
"Setting an empty password for user {0}. This is dangerous and totally "
"insecure!"
msgstr ""

#: core/users.py:493
msgid "Set password for user {0} to {1}."
msgstr ""

#: core/users.py:500
msgid "Changed password for user {0}."
msgstr ""

#: core/users.py:572
msgid "Account {0} already locked."
msgstr ""

#: core/users.py:582
msgid "Locked user account {0}."
msgstr ""

#: core/users.py:595
msgid "Unlocked user account {0}."
msgstr ""

#: core/users.py:600
msgid "Account {0} already unlocked."
msgstr ""

#: core/users.py:695 core/groups.py:975
#, python-format
msgid "Backend %s does not exist or is not enabled."
msgstr ""

#: core/users.py:701 core/groups.py:981
msgid "Skipped move of group {0}, already stored in backend {1}."
msgstr ""

#: core/users.py:708 core/groups.py:988
msgid ""
"Skipped move of restricted system group {0} (please use {1} if you really "
"want to do this, but it is strongly not recommended)."
msgstr ""

#: core/users.py:716
msgid ""
"Cannot move an associated system group without moving its standard  group "
"too. Please move the standard group instead, if this is what you meant."
msgstr ""

#: core/users.py:725 core/groups.py:1005
msgid ""
"Skipped move of group {0} to backend {1} because move of associated "
"responsible system group failed."
msgstr ""

#: core/users.py:738 core/groups.py:1018
msgid ""
"Skipped move of group {0} to backend {1} because move of associated system "
"guest group failed."
msgstr ""

#: core/users.py:748
msgid ""
"Exception {0} happened while trying to move group {1} from {2} to {3}, "
"aborting (group left unchanged)."
msgstr ""

#: core/users.py:770
msgid ""
"Exception {0} happened while trying to restore a stable situation during "
"group {1} move, we could be in big trouble."
msgstr ""

#: core/users.py:780
msgid "Moved group {0} from backend {1} to {2}."
msgstr ""

#: core/users.py:808
msgid "Invalid skel \"{0}\". Valid skels are {1}."
msgstr ""

#: core/users.py:848
msgid "Applyed skel {0} for user {1}"
msgstr ""

#: core/users.py:947
msgid "Auto-cleaned {0}'s gecos to \"{1}\"."
msgstr ""

#: core/users.py:952
#, python-format
msgid "%s Automatic Backup System"
msgstr ""

#: core/users.py:953
msgid "Special Executable Restrictor Account"
msgstr ""

#: core/users.py:954
msgid "ISC Domain Name Service daemon"
msgstr ""

#: core/users.py:955
msgid "Apple Calendar Server Daemon"
msgstr ""

#: core/users.py:956
msgid "Generic System Daemon"
msgstr ""

#: core/users.py:957
msgid "DHCP Daemon"
msgstr ""

#: core/users.py:958
msgid "DNSmasq (DHCP & DNS) Daemon"
msgstr ""

#: core/users.py:959
msgid "FTP Daemon / restricted account"
msgstr ""

#: core/users.py:960
msgid "Games Specific Account"
msgstr ""

#: core/users.py:961
msgid "Hardware Abstraction Layer Daemon"
msgstr ""

#: core/users.py:962
msgid "Hewlett-Packard Linux Imaging & Printing Daemon"
msgstr ""

#: core/users.py:963
msgid "Internet Relay Chats Daemon"
msgstr ""

#: core/users.py:964
msgid "Kernel OOPs Tracking Daemon"
msgstr ""

#: core/users.py:965
msgid "Canonical's Lanscape Remote Administration account"
msgstr ""

#: core/users.py:966
msgid "UUID Library Account"
msgstr ""

#: core/users.py:967
msgid "Line Printer Daemon"
msgstr ""

#: core/users.py:968
msgid "Local E-Mail System Account"
msgstr ""

#: core/users.py:969
msgid "Unix Manual Pages Generator"
msgstr ""

#: core/users.py:970
msgid "Memory Cache Daemon"
msgstr ""

#: core/users.py:971 core/users.py:975
msgid "System Message Bus (D-BUS)"
msgstr ""

#: core/users.py:972
msgid "Video Motion Detection daemon"
msgstr ""

#: core/users.py:973 core/users.py:974
msgid "Very unprivileged System Account"
msgstr ""

#: core/users.py:976
msgid "PostfixÂ® Mail System"
msgstr ""

#: core/users.py:977
msgid "Web (& possibly more) Proxy Server"
msgstr ""

#: core/users.py:978
msgid "John Root, System Administrator (BOFH)"
msgstr ""

#: core/users.py:979
msgid "Scanner Access Now Easy daemon"
msgstr ""

#: core/users.py:980
msgid "Secure SHell Daemon"
msgstr ""

#: core/users.py:981
msgid "Generic Synchronization Account"
msgstr ""

#: core/users.py:982
msgid "Generic System Account"
msgstr ""

#: core/users.py:983
msgid "System Logger Daemon"
msgstr ""

#: core/users.py:984
msgid "Unix to Unix Copy Protocol daemon"
msgstr ""

#: core/users.py:985
msgid "Web Server unprivileged Account"
msgstr ""

#: core/users.py:986
msgid "Network Time Protocol Daemon"
msgstr ""

#: core/users.py:990
msgid ""
"{0}'s GECOS \"{1}\" is currently meaningless, or we have a far better and "
"hypish value \"{2}\" ready to enhance it. Would you like to use it?"
msgstr ""

#: core/users.py:1006
#, python-format
msgid "Checking system account %sâ¦"
msgstr ""

#: core/users.py:1010
msgid "Account {0} already beiing ckecked, aborting."
msgstr ""

#: core/users.py:1056
#, python-format
msgid "Checking restricted system account %sâ¦"
msgstr ""

#: core/users.py:1060
msgid "account {0} already beiing ckecked, aborting."
msgstr ""

#: core/users.py:1101 core/groups.py:1674
msgid "(not watched)"
msgstr ""

#: core/users.py:1116
msgid "âÂ no membershipÂ â"
msgstr ""

#: core/users.py:1189 core/configuration.py:685
msgid "user"
msgstr ""

#: core/users.py:1192
msgid "UID"
msgstr ""

#: core/users.py:1282
msgid "{0}.reload: Overwritten uid {1}."
msgstr ""

#: core/users.py:1295
msgid "{0}: removing disapeared user {1}."
msgstr ""

#: core/users.py:1302
msgid "Reloading {0} controller too."
msgstr ""

#: core/users.py:1372
msgid "UID {0} does not exist."
msgstr ""

#: core/users.py:1381
msgid ""
"Specified directory {0} for system user {1} already exists. If you really "
"want to use it, please use the --force argument."
msgstr ""

#: core/users.py:1393
msgid ""
"Specified home directory {0} for system user {1} is outside {2} and /var, or "
"inside {3} or a temporary directory (/var/tmp, /tmp). This is unsupported, "
"Aborting."
msgstr ""

#: core/users.py:1404
msgid ""
"Specified home directory {0} for system user {1} is already owned by another "
"user. Please choose another one."
msgstr ""

#: core/users.py:1413
msgid ""
"Specifying an alternative home directory is not allowed for standard users. "
"Using standard home path {0} instead."
msgstr ""

#: core/users.py:1426
msgid ""
"You must specify a login, a firstname *and* lastname couple, or a GECOS "
"(login will be built from them)."
msgstr ""

#: core/users.py:1443
#, python-format
msgid "User account %s"
msgstr ""

#: core/users.py:1450
msgid ""
"You must specify a lastname and a firstname *or* a GECOS. If you specify the "
"GECOS, do not specify firstname/lastname."
msgstr ""

#: core/users.py:1465
msgid "Malformed login \"{0}\", must match /{1}/."
msgstr ""

#: core/users.py:1491
msgid "Invalid shell \"{0}\". Valid shells are: {1}."
msgstr ""

#: core/users.py:1498
msgid "Invalid skel \"{0}\". Valid skels are: {1}."
msgstr ""

#: core/users.py:1511
msgid "UID {0} is already taken by user {1}. Please choose another one."
msgstr ""

#: core/users.py:1520
msgid "User account {0} already exists!"
msgstr ""

#: core/users.py:1523
msgid ""
"A user account {0} already exists but has not the same type. Please choose "
"another login for your user."
msgstr ""

#: core/users.py:1550
msgid ""
"A group named {0} exists on the system. This could eventually conflict in "
"Debian/Ubuntu system tools. Please choose another login, or use the --force "
"argument if you really want to add this user on the system."
msgstr ""

#: core/users.py:1568
msgid "Autogenerated UID for user {0}: {1}."
msgstr ""

#: core/users.py:1576
msgid ""
"UID out of range for the kind of user account you specified. System UIDs "
"must be between {0} and {1}, standard UIDs must be between {2} and {3}."
msgstr ""

#: core/users.py:1696
msgid "Autogenerated password for user {0} (uid={1}): {2}."
msgstr ""

#: core/users.py:1705
msgid "Created {accttype} user {login} (uid={uid})."
msgstr ""

#: core/users.py:1706 core/_controllers.py:535
msgid "system"
msgstr ""

#: core/users.py:1706
msgid "standard"
msgstr ""

#: core/users.py:1755
#, python-format
msgid ""
"Cannot delete restricted system account %s without the --force argument, "
"this is too dangerous."
msgstr ""

#: core/users.py:1783
msgid "{0}: account {1} already not referenced in controller!"
msgstr ""

#: core/users.py:1810
msgid "Deleted user account {0}."
msgstr ""

#: core/users.py:1958
msgid "Some user(s) check(s) did not pass, or were not corrected."
msgstr ""

#: core/users.py:1970
msgid "You must specify an UID or a login to test existence of."
msgstr ""

#: core/users.py:1979
#, python-format
msgid "User %s does not exist"
msgstr ""

#: core/users.py:1987
#, python-format
msgid "UID %s does not exist"
msgstr ""

#: core/backends/shadow.py:76
msgid ""
"{0} shadow backend (this is important, please do not try to set {1} in {2}!"
msgstr ""

#: core/backends/shadow.py:78
msgid "RE-enabled"
msgstr ""

#: core/backends/shadow.py:209
msgid "{0}: added missing entry for user {1} in {2}."
msgstr ""

#: core/backends/shadow.py:236
msgid "{0}: cleaned users data rewrite requested in the background."
msgstr ""

#: core/backends/shadow.py:287
msgid "{0}: skipped line \"{1}\" from {2}, it seems corrupt."
msgstr ""

#: core/backends/shadow.py:341
msgid "{0}: added missing record for group {1} in {2}."
msgstr ""

#: core/backends/shadow.py:359
msgid "{0}: cleaned groups data rewrite requested in the background."
msgstr ""

#: core/backends/shadow.py:435
msgid "{0}: saved users data to disk."
msgstr ""

#: core/backends/shadow.py:503
msgid "{0}: saved groups data to disk."
msgstr ""

#: core/backends/shadow.py:532 core/backends/dnsmasq.py:176
msgid "{0}: configuration file {1} changed, reloading {2} controller."
msgstr ""

#: core/backends/jsonfile.py:81
msgid "{0}: Exception while loading tasks"
msgstr ""

#: core/backends/jsonfile.py:83
msgid "{0}: consequently, THERE IS CURRENTLY NO TASKS IN THE DAEMON!"
msgstr ""

#: core/backends/jsonfile.py:127
msgid "{0}: could not save tasks on disk"
msgstr ""

#: core/backends/sqlite.py:20
msgid "SqliteBackend is not yet implemented."
msgstr ""

#: core/backends/pickle.py:20
msgid "PickleBackend is not yet implemented."
msgstr ""

#: core/backends/openldap.py:110
msgid ""
"{0}: Problem initializing the backend. OpenLDAP seems installed but not "
"configured, or unusable. Please run {1} to try fix the situation."
msgstr ""

#: core/backends/openldap.py:183
msgid "{0}: setting a default password in {1}."
msgstr ""

#: core/backends/openldap.py:242
msgid ""
"{0}: server {1} is down, disabling backend. This will produce timeouts "
"because {2} is enabled in NSS configuration."
msgstr ""

#: core/backends/openldap.py:323
msgid "{0}: checking backend configurationâ¦"
msgstr ""

#: core/backends/openldap.py:329
msgid "{0:s}: backend not available, not checking."
msgstr ""

#: core/backends/openldap.py:333
msgid "{0}: you must be root or have {1} access to continue."
msgstr ""

#: core/backends/openldap.py:402
msgid "{0}: checking slapd schemataâ¦"
msgstr ""

#: core/backends/openldap.py:435
msgid "{0}: disabling AppArmor profile for {1}."
msgstr ""

#: core/backends/openldap.py:443
msgid "{0:s}: Exception encountered while disabling AppArmor profile."
msgstr ""

#: core/backends/openldap.py:456
msgid "{0}: re-enabling AppArmor profile for {1}."
msgstr ""

#: core/backends/openldap.py:464
msgid "{0:s}: Exception encountered while re-enabling AppArmor profile."
msgstr ""

#: core/backends/openldap.py:471
msgid "{0}: {1} lacks mandatory schema {2}."
msgstr ""

#: core/backends/openldap.py:480
msgid "{0}: loading schema {1} into {2}."
msgstr ""

#: core/backends/openldap.py:488
msgid "{0}: adding {1} -> {2} into schema {3}."
msgstr ""

#: core/backends/openldap.py:495
msgid "{0}: loaded schema {1} into {2}."
msgstr ""

#: core/backends/openldap.py:501
msgid "{0}: skipped already present dn {1}."
msgstr ""

#: core/backends/openldap.py:505
msgid "{0}: Exception encountered while adding {1} > {2} into schema {3}."
msgstr ""

#: core/backends/openldap.py:514 core/backends/openldap.py:566
msgid "Cannot continue without altering slapd configuration."
msgstr ""

#: core/backends/openldap.py:527
msgid "{0}: {1} schema {2} needs updating."
msgstr ""

#: core/backends/openldap.py:536
msgid "{0}: updating schema {1} in {2}."
msgstr ""

#: core/backends/openldap.py:551
msgid "{0}: updated schema {1} in {2}."
msgstr ""

#: core/backends/openldap.py:557
msgid ""
"{0}: Exception encountered while modifying {1} from {2} TO {3} in schema {4}"
msgstr ""

#: core/backends/openldap.py:574
msgid "{0}: checking system filesâ¦"
msgstr ""

#: core/backends/openldap.py:600
msgid ""
"{0}: {1} should contain your LDAP system password, but is currently empty. "
"Fill it with a random password?"
msgstr ""

#: core/backends/openldap.py:612
msgid ""
"{0}: autogenerated password for LDAP user {1}: \"{2}\". It is up to you to "
"update the LDAP database with it, now."
msgstr ""

#: core/backends/openldap.py:633
msgid ""
"{0}: {1} is mandatory for {2} to work properly. Cannot continue without "
"this, sorry!"
msgstr ""

#: core/backends/openldap.py:664
msgid "{0}: binding in EXTERNAL SASL mode."
msgstr ""

#: core/backends/openldap.py:673 core/backends/openldap.py:697
msgid "{0}: binding as as {1}."
msgstr ""

#: core/backends/openldap.py:816
msgid "{0}: skipped account {1}"
msgstr ""

#: core/backends/openldap.py:872
msgid "{0}: saved all user accounts in the directory."
msgstr ""

#: core/backends/openldap.py:885
msgid "{0}: saved all groups in the directory."
msgstr ""

#: core/backends/openldap.py:963
msgid "{0}: unknown mode {1} for user {2}(uid={3})."
msgstr ""

#: core/backends/openldap.py:968
msgid "{0}: unable to save user {1}."
msgstr ""

#: core/backends/openldap.py:973
msgid "{0}: saved user account {1} in the directory."
msgstr ""

#: core/backends/openldap.py:1035
msgid "{0}: unknown mode {1} for group {2}(gid={3})."
msgstr ""

#: core/backends/openldap.py:1042
msgid "{0}: unable to save group {1}."
msgstr ""

#: core/backends/openldap.py:1047
msgid "{0}: saved group {1} in the directory."
msgstr ""

#: core/backends/openldap.py:1064
msgid "{0}: deleted user account {1} from the directory."
msgstr ""

#: core/backends/openldap.py:1082
msgid "{0}: deleted group {1} from the directory."
msgstr ""

#: core/privileges.py:47
msgid "privilege"
msgstr ""

#: core/privileges.py:50 core/groups.py:1841
msgid "GID"
msgstr ""

#: core/privileges.py:143
msgid "Skipped privilege {0} refering to a non-existing group."
msgstr ""

#: core/privileges.py:163
msgid ""
"Triggering privilege whitelist rewrite in 3 seconds to remove non-existing "
"references or duplicates."
msgstr ""

#: core/privileges.py:189
#, python-format
msgid "Skipped privilege %s, already whitelisted."
msgstr ""

#: core/privileges.py:194
#, python-format
msgid "Group %s must not be promoted to privilege for security reasons."
msgstr ""

#: core/privileges.py:200
#, python-format
msgid ""
"Group %s cannot be promoted to privilege, it is not a pure system group."
msgstr ""

#: core/privileges.py:211
#, python-format
msgid "Added privilege %s to whitelist."
msgstr ""

#: core/privileges.py:226
#, python-format
msgid "Removed privilege %s from whitelist."
msgstr ""

#: core/privileges.py:231
#, python-format
msgid "Skipped privilege %s, already not present in the whitelist."
msgstr ""

#: core/privileges.py:249
msgid "Privilege {0} does not exist"
msgstr ""

#: core/remote.py:65
msgid "Exception while running {0}({1}, {2})"
msgstr ""

#: core/remote.py:73
msgid "{0}: Pickle error on licornd side, passing this turn."
msgstr ""

#: core/remote.py:85
msgid "Pyro connection closed, retrying{0}."
msgstr ""

#: core/remote.py:86
msgid " forever"
msgstr ""

#: core/remote.py:105
msgid "Exception while releasing LMC."
msgstr ""

#: core/system.py:231
msgid "Already shutting down!"
msgstr ""

#: core/system.py:261
msgid "Warning: automatic system shutdown"
msgstr ""

#: core/system.py:262
msgid "Accept fate"
msgstr ""

#: core/system.py:263
msgid "Cancel shutdown"
msgstr ""

#: core/system.py:264
msgid ""
"The system will automatically shutdown in one minute. Please save all your "
"work and close your session.\\n\\nYou can choose to cancel this operation, "
"but i imagine that your sysadmin will not be happy 'bout that!"
msgstr ""

#: core/system.py:295
msgid "Already NOT currently shutting down."
msgstr ""

#: core/system.py:316
msgid "{0:s}: machine {1:s} wants us to restart (condition={2:s}, delay={3:s})"
msgstr ""

#: core/__init__.py:39
msgid ""
"The daemon did not wake us in {0} seconds, there is probably a problem with "
"it. Please check the log for errors or contact your system administrator if "
"it is not you. Else you already know you are in trouble ;-)"
msgstr ""

#: core/__init__.py:46
msgid "Connection established. Please be patient, the daemon seems quite busy."
msgstr ""

#: core/__init__.py:59
msgid "OK. We're running now!"
msgstr ""

#: core/__init__.py:210
msgid "Server backends: {0}"
msgstr ""

#: core/__init__.py:397
#, python-format
msgid "trying to connect to server %s."
msgstr ""

#: core/__init__.py:480
msgid "Your user account is not allowed to connect to the LicornÂ® daemon."
msgstr ""

#: core/__init__.py:486
#, python-format
msgid ""
"Cannot connect to the daemon, but it has been successfully launched. I  "
"suspect you are in trouble (was: %s)"
msgstr ""

#: core/__init__.py:491
#, python-format
msgid ""
"Cannot reach our daemon at %s, retrying in 5 seconds. Check your network "
"connection, cable, DNS and firewall. Perhaps  the LicornÂ® server is simply "
"down."
msgstr ""

#: core/__init__.py:527
msgid "Waiting up to {0} seconds for daemon to come upâ¦ Please hold on."
msgstr ""

#: core/configuration.py:107
#, python-format
msgid "Configuration initialization failed: %s"
msgstr ""

#: core/configuration.py:164
#, python-format
msgid ""
"Temporary directory %s has vanished during run, or already been wiped by "
"another process."
msgstr ""

#: core/configuration.py:284
msgid ""
"Cannot create or change owner of {0}[/data]:\n"
"\t{1}"
msgstr ""

#: core/configuration.py:337
msgid ""
"You are running an unknown or unsupported non-LSB but POSIX Operating "
"System. Please get in touch with {0} developpers, and consider submitting a "
"patch to make your system officially supported."
msgstr ""

#: core/configuration.py:361
msgid ""
"Your Ubuntu version \"{0}\" is no longer supported by {1} developpers nor "
"Canonical. You should consider upgrading to a newer one."
msgstr ""

#: core/configuration.py:370
msgid ""
"Greetings old-timer :-) Does {0} still run on your aged Ubuntu version "
"\"{1}\"? Anyway it is no longer supported by {0} developpers. Even Canonical "
"nor Ubuntu community does not support it anymore. You should very strongly "
"consider upgrading to a newer version."
msgstr ""

#: core/configuration.py:381
msgid "This Ubuntu version \"{0}\" is not [yet] supported, sorry !"
msgstr ""

#: core/configuration.py:398
msgid ""
"Your Debian version (\"{0}\") is no longer supported by {1} developpers nor "
"Debian. You should consider upgrading to a newer one."
msgstr ""

#: core/configuration.py:407
msgid ""
"Greetings old-timer :-) Does {0} still run on your aged Debian version "
"\"{1}\"? Anyway it is no longer supported by {0} developpers. Even Debian "
"community does not support it anymore. You should very strongly consider "
"upgrading to a newer version."
msgstr ""

#: core/configuration.py:418
msgid "This Debian version \"{0}\" is not [yet] supported, sorry !"
msgstr ""

#: core/configuration.py:423
msgid ""
"Your LSB distro is not yet supported. This should be trivial to implement, "
"but this is still to be done, sorry!"
msgstr ""

#: core/configuration.py:469
msgid "MTA not installed or unsupported."
msgstr ""

#: core/configuration.py:528
msgid ""
"Mail{box,dir} system not supported yet. Please get in touch with dev@licorn."
"org."
msgstr ""

#: core/configuration.py:569
msgid ""
"Custom skels must have at least {0} perms on dirs and {1} on files:\n"
"\t{2}"
msgstr ""

#: core/configuration.py:686
msgid "users"
msgstr ""

#: core/configuration.py:706 core/groups.py:1838 daemon/rwi.py:929
#: daemon/rwi.py:942
msgid "group"
msgstr ""

#: core/configuration.py:707
msgid "groups"
msgstr ""

#: core/configuration.py:780
msgid "In {0}, directive {1} should be at least {2}, but it is {3}."
msgstr ""

#: core/configuration.py:790
msgid "In {0}, directive {1} should be set to {2}, but it is {3}."
msgstr ""

#: core/configuration.py:801
msgid "In {0}, directive {1} is missing. Setting it to {2}."
msgstr ""

#: core/configuration.py:811
msgid "{0} lacks mandatory configuration directive(s)."
msgstr ""

#: core/configuration.py:819 core/configuration.py:918
msgid "Altered {0} to match LicornÂ® pre-requisites."
msgstr ""

#: core/configuration.py:824 core/configuration.py:924
#, python-format
msgid ""
"Insufficient permissions. Are you root?\n"
"\t%s"
msgstr ""

#: core/configuration.py:828
#, python-format
msgid ""
"Modifications in %s are mandatory for LicornÂ® to work properly with other "
"system tools (adduser/useradd). Cannot continue without this, sorry!"
msgstr ""

#: core/configuration.py:880
#, python-format
msgid "%s does not exist on this system!"
msgstr ""

#: core/configuration.py:890
msgid "In {0}, directive {1} should be {2}, but it is {3}."
msgstr ""

#: core/configuration.py:901
msgid "In {0}, directive {1} is not present but should be, with value {2}."
msgstr ""

#: core/configuration.py:910
msgid "{0} should be altered to work with LicornÂ®. Fix it?"
msgstr ""

#: core/configuration.py:929
#, python-format
msgid ""
"Modifications in %s are mandatory for LicornÂ® to work properly. Cannot "
"continue without this, sorry!"
msgstr ""

#: core/configuration.py:974
#, python-format
msgid "Bad CLI output format \"%s\"!"
msgstr ""

#: core/configuration.py:978
msgid "Application name"
msgstr ""

#: core/configuration.py:979
msgid "Storage backends for Core objects"
msgstr ""

#: core/configuration.py:980
msgid "Application main configuration directory"
msgstr ""

#: core/configuration.py:981
msgid "Application extensions (available add-ons)"
msgstr ""

#: core/configuration.py:982
msgid "This current help"
msgstr ""

#: core/configuration.py:983
msgid "Application main configuration filename"
msgstr ""

#: core/configuration.py:984
msgid "System groups tagged as privileges"
msgstr ""

#: core/configuration.py:985
msgid "System valid shells when creating a new user"
msgstr ""

#: core/configuration.py:986
msgid "System valid skels to apply on a user account"
msgstr ""

#: core/configuration.py:987
msgid "Mandatory system groups for {0}"
msgstr ""

#: core/configuration.py:993
#, python-format
msgid "Sorry, \"%s\" is not a recognized keyword."
msgstr ""

#: core/configuration.py:998
msgid "Valid keywords for {0} (fuzzy matching):"
msgstr ""

#: core/configuration.py:1002 core/machines.py:1225 core/_modules.py:654
#: extensions/volumes.py:450 extensions/volumes.py:502
#: extensions/caldavd.py:192 extensions/caldavd.py:213
#: extensions/caldavd.py:458 extensions/caldavd.py:492
#: extensions/caldavd.py:529 extensions/caldavd.py:584
#: extensions/caldavd.py:615 extensions/caldavd.py:649
#: extensions/caldavd.py:677
msgid "{0}: {1}"
msgstr ""

#: core/configuration.py:1168
msgid ""
"New hostname must be composed only of letters, digits and hyphens, but not "
"starting nor ending with an hyphen !"
msgstr ""

#: core/configuration.py:1180
#, python-format
msgid ""
"Cannot modify /etc/hostname, verify the file is still clean:\n"
"\t%s)"
msgstr ""

#: core/configuration.py:1283 extensions/samba3/samba3.py:246
#, python-format
msgid ""
"Filesystem must be mounted with \"acl\" option:\n"
"\t%s"
msgstr ""

#: core/configuration.py:1354
#, python-format
msgid "Checking existence of group %sâ¦"
msgstr ""

#: core/configuration.py:1360
#, python-format
msgid ""
"The system group %s is mandatory for the system to work properly, but it "
"does not exist yet."
msgstr ""

#: core/configuration.py:1376
msgid ""
"The system group \"{0}\" is mandatory but does not exist on your system !\n"
"Use \"chk config\" or \"add group --system --name {0}\" to fix the problem "
"before continuing."
msgstr ""

#: core/configuration.py:1413
msgid "/etc/hosts must have at least o+r permissions."
msgstr ""

#: core/configuration.py:1419
msgid "Current hostname and /etc/hostname are not in sync!"
msgstr ""

#: core/configuration.py:1428
msgid ""
"Hostname {0} does not resolve to 127.0.0.1 but to {1}, please check /etc/"
"hosts!"
msgstr ""

#: core/configuration.py:1439
#, python-format
msgid ""
"127.0.0.1 does not resolve back to hostname %s, please check /etc/hosts and/"
"or DNS !"
msgstr ""

#: core/configuration.py:1483
msgid ""
"Cannot resolve {0} ({1}), is the DNS server running on 127.0.0.1? Else check "
"DNS files syntax !"
msgstr ""

#: core/configuration.py:1490
msgid "127.0.0.1 *must* be in /etc/resolv.conf on a LicornÂ® server."
msgstr ""

#: core/configuration.py:1496
msgid ""
"Problem while resolving {0}, please check your configuration:\n"
"\terrno {1}, {2}"
msgstr ""

#: core/configuration.py:1511
msgid ""
"127.0.0.1 *must* be the only nameserver in /etc/resolv.conf on a LicornÂ® "
"server."
msgstr ""

#: core/configuration.py:1516
#, python-format
msgid ""
"DNS does not seem properly configured (%s does not resolve back to itself)."
msgstr ""

#: core/configuration.py:1521
msgid ""
"DNS is not properly configured ({0} does not resolve to {1}, but to {2})."
msgstr ""

#: core/configuration.py:1529
msgid ""
"127.0.0.1 should be the only nameserver in /etc/resolv.conf on a LicornÂ® "
"server."
msgstr ""

#: core/_objects.py:150
msgid ""
"Cannot set a backend without a \"move_to_backend\" method on the current "
"instance."
msgstr ""

#: core/_objects.py:254
msgid "Inotify state {0} for {1} {2}."
msgstr ""

#: core/_objects.py:255 core/groups.py:451
msgid "unchanged"
msgstr ""

#: core/_objects.py:264 core/groups.py:458 extensions/simplesharing.py:427
msgid "activated"
msgstr ""

#: core/_objects.py:265
msgid "setup"
msgstr ""

#: core/_objects.py:275 core/groups.py:461 extensions/simplesharing.py:427
msgid "deactivated"
msgstr ""

#: core/_objects.py:276
msgid "torn down"
msgstr ""

#: core/_objects.py:293
msgid ""
"Switched {0} {1} inotify state to {2} (watchers for shared content are "
"beiing {3} in the background, this can take a whileâ¦)"
msgstr ""

#: core/_objects.py:568
msgid "Watch add failed for {0}, reason {1}"
msgstr ""

#: core/_objects.py:572
msgid "{0}: overwriting watch {1}!"
msgstr ""

#: core/_objects.py:599
msgid "Cannot remove watch {0}, continuing"
msgstr ""

#: core/_objects.py:616 core/_objects.py:633
msgid "{0}.__unwatch_directory in {1}: {2} not found in watched dirs."
msgstr ""

#: core/_objects.py:702
msgid ""
"While deleting inotifier watches for {0}, it has never been checked (this is "
"harmless)."
msgstr ""

#: core/_objects.py:762
msgid "{0} {1}: somebody is already checking; operation aborted."
msgstr ""

#: core/_objects.py:768
msgid "shared"
msgstr ""

#: core/_objects.py:768
msgid "home"
msgstr ""

#: core/_objects.py:777
msgid "Checking {0} {1}â¦"
msgstr ""

#: core/_objects.py:794
msgid "{0} directory {1} already exists."
msgstr ""

#: core/_objects.py:798
msgid "{0} directory {1} does not exist but it is mandatory. Create it?"
msgstr ""

#: core/_objects.py:806
msgid "Created {0} directory {1}."
msgstr ""

#: core/_objects.py:811
#, python-format
msgid "Directory %s does not exist but is mandatory. Check aborted."
msgstr ""

#: core/_objects.py:847
msgid "{0} {1}: meta-data changed on path {2}."
msgstr ""

#: core/_objects.py:858
msgid "{0} {1}: meta-data changed on {2} path(s)."
msgstr ""

#: core/_objects.py:867
msgid "{0} {1}: no shared data to check."
msgstr ""

#: core/_objects.py:873
msgid "{0} {1}: {2}"
msgstr ""

#: core/_objects.py:968
msgid ""
"{0}: home directory {1} disappeared. If this is intentional, do not forget "
"to run \"{2}\" afterwards, to restore the inotifier watch."
msgstr ""

#: core/_objects.py:980
msgid "{0}: {1} called on an external path {2}, aborting!"
msgstr ""

#: core/_objects.py:1042
msgid "{0}: problem while checking {1}, aborting"
msgstr ""

#: core/tasks.py:159
msgid "Monday"
msgstr ""

#: core/tasks.py:160
msgid "Tuesday"
msgstr ""

#: core/tasks.py:161
msgid "Wednesday"
msgstr ""

#: core/tasks.py:162
msgid "Thrusday"
msgstr ""

#: core/tasks.py:163
msgid "Friday"
msgstr ""

#: core/tasks.py:164
msgid "Satursday"
msgstr ""

#: core/tasks.py:165
msgid "Sunday"
msgstr ""

#: core/tasks.py:166
msgid "ALL"
msgstr ""

#: core/tasks.py:275
msgid ""
"{0}: arg {1} cannot be resolved, (error: {2}), you may want to defer the "
"action parameters resolution using the {3} option."
msgstr ""

#: core/tasks.py:283 core/tasks.py:292
msgid "Exception while evaluating \"{0}\""
msgstr ""

#: core/tasks.py:313
msgid "next @{0} {1}"
msgstr ""

#: core/tasks.py:315
#, python-format
msgid "(in %s)"
msgstr ""

#: core/tasks.py:322
msgid "Operation thread is down."
msgstr ""

#: core/tasks.py:325
msgid ""
"{id}: {name}, {counter}, {next}\n"
"\t\t\t{action}({args}{comma}{kwargs}) {defer}\n"
"\t\t\t{time}"
msgstr ""

#: core/tasks.py:335
msgid " ({0})"
msgstr ""

#: core/tasks.py:336
msgid "with deferred resolution"
msgstr ""

#: core/tasks.py:340
msgid "ran {0} time(s)"
msgstr ""

#: core/tasks.py:466
msgid "Error while checking argument \"{0}\" against {1}:"
msgstr ""

#: core/tasks.py:637
msgid "{0}: {1} for task {2}, un-scheduled."
msgstr ""

#: core/tasks.py:639
msgid "No remaining occurence"
msgstr ""

#: core/tasks.py:695
msgid "{1} on task {0} for argument {2}={5}: should be between {3} and {4}"
msgstr ""

#: core/tasks.py:698
msgid "Not in range"
msgstr ""

#: core/tasks.py:727
msgid ""
"{0} on task {1} for argument {2}={3}: excepted type are String, Integer or "
"None"
msgstr ""

#: core/tasks.py:730 core/tasks.py:776 core/tasks.py:804 core/tasks.py:811
msgid "Type error"
msgstr ""

#: core/tasks.py:773
msgid "{0} on task {1} for argument {2}={3} has to be an Integer"
msgstr ""

#: core/tasks.py:783
msgid "{0} on task {1} for argument {2}={3} has to be > {4}"
msgstr ""

#: core/tasks.py:786
msgid "Argument error"
msgstr ""

#: core/tasks.py:795
msgid "{1} on task {0} for argument {2}={3}: should be transtypable to int"
msgstr ""

#: core/tasks.py:797
msgid "Type Error"
msgstr ""

#: core/tasks.py:802
msgid "{0} on task {1} for argument 'action'={2} has to be a String"
msgstr ""

#: core/tasks.py:809
msgid "{0} on task {1} for argument 'defer_resolution'={2} has to be a Boolean"
msgstr ""

#: core/tasks.py:818
msgid ""
"{0} on extinction task {1} for argument 'hour'={2} has to be a Integer where "
"0<=hour<60"
msgstr ""

#: core/tasks.py:820 core/tasks.py:825 core/tasks.py:831
msgid "BadArgumentError"
msgstr ""

#: core/tasks.py:823
msgid ""
"{0} on extinction task {1} for argument 'minute'={2} has to be a Integer "
"where 0<=minute<60"
msgstr ""

#: core/tasks.py:829
msgid ""
"{0} on extinction task {1} for argument 'week_day'={2} has to be a Integer "
"where 0<=week_day<7"
msgstr ""

#: core/tasks.py:873
msgid ""
"Another rule already exists for the machine {0} on day {1} at {2}, please "
"delete it first."
msgstr ""

#: core/tasks.py:955
msgid "{0}: Exception while reloading tasks"
msgstr ""

#: core/tasks.py:991
msgid "Another task with the same name already exists"
msgstr ""

#: core/tasks.py:1015
msgid "{0}: addition prevented: {1}"
msgstr ""

#: core/tasks.py:1030
msgid "{0}: {1} for task {2}: {3} argument cannot be resolved."
msgstr ""

#: core/tasks.py:1033
msgid "Cannot resolve action"
msgstr ""

#: core/tasks.py:1049
msgid "{0}: scheduling task {1} on {2}: {3}; {4}"
msgstr ""

#: core/tasks.py:1077
msgid "{0}: deletion prevented: {1}"
msgstr ""

#: core/tasks.py:1093
msgid "{0}: task {1} unscheduled and {2}."
msgstr ""

#: core/tasks.py:1096
msgid "removed"
msgstr ""

#: core/tasks.py:1113
msgid "Scheduler threads:"
msgstr ""

#: core/tasks.py:1115
msgid ""
"\n"
"\t{0} (id: {1}): {2} > {3}(args={4}, kwargs={5})"
msgstr ""

#: core/tasks.py:1117
#, python-format
msgid "next in %s "
msgstr ""

#: core/tasks.py:1120
msgid "Currently running"
msgstr ""

#: core/tasks.py:1129
msgid "task"
msgstr ""

#: core/tasks.py:1132
msgid "id"
msgstr ""

#: core/groups.py:100
msgid ""
"Cannot build a valid UNIX group name (got {0}, which does not verify {1}) "
"with the string you provided \"{2}\"."
msgstr ""

#: core/groups.py:425
msgid ""
"group {0}: Cannot set a profile ({1}) which has not the same GID as me ({2} !"
"= {3})!"
msgstr ""

#: core/groups.py:449
msgid "Permissive state {0} for group {1}."
msgstr ""

#: core/groups.py:477
msgid ""
"Switched group {0} permissive state to {1} (Shared content permissions are "
"beiing checked in the background, this can take a whileâ¦)"
msgstr ""

#: core/groups.py:496
msgid "You must specify a description"
msgstr ""

#: core/groups.py:500
msgid "Malformed description \"{0}\", must match /{1}/i."
msgstr ""

#: core/groups.py:513
msgid "Changed group {0} description to \"{1}\"."
msgstr ""

#: core/groups.py:525
msgid "You must specify a groupSkel"
msgstr ""

#: core/groups.py:528 core/groups.py:2258
msgid "Invalid skel {0}. Valid skels are: {1}."
msgstr ""

#: core/groups.py:539
msgid "Changed group {0} skel to {1}."
msgstr ""

#: core/groups.py:739
msgid "group {0}: removed relationship for non-existing user {1}."
msgstr ""

#: core/groups.py:747
msgid "group {0}: removed duplicate relationship for member {1}."
msgstr ""

#: core/groups.py:754
msgid "group {0}: removed primary member {1} from auxilliary members list."
msgstr ""

#: core/groups.py:782 core/groups.py:883
msgid "You must specify a users list"
msgstr ""

#: core/groups.py:809
msgid "User {0} is already a member of {1} (primary or not), skipped."
msgstr ""

#: core/groups.py:844
msgid "Added user {0} to members of group {1}."
msgstr ""

#: core/groups.py:910
msgid "Removed user {0} from members of group {1}."
msgstr ""

#: core/groups.py:931
msgid "Skipped user {0}, already not a member of group {1}"
msgstr ""

#: core/groups.py:996
msgid ""
"Cannot move an associated system group without moving its standard group "
"too. Please move the standard group instead, if this is what you meant."
msgstr ""

#: core/groups.py:1029
msgid ""
"Exception happened while trying to move group {0} from {1} to {2}, aborting "
"(group left unchanged)"
msgstr ""

#: core/groups.py:1052
msgid ""
"An error occured while trying to restore a stable situation during group {0} "
"move, we could be in big trouble."
msgstr ""

#: core/groups.py:1065
msgid "Moved group {0} from {1} to {2}."
msgstr ""

#: core/groups.py:1115
#, python-format
msgid "Checking %s symlinks in members homes, this can be longâ¦"
msgstr ""

#: core/groups.py:1155 core/groups.py:1636
#, python-format
msgid "Deleted symlink %s."
msgstr ""

#: core/groups.py:1160 core/groups.py:1640
msgid "Unable to delete symlink {0} (was: {1})."
msgstr ""

#: core/groups.py:1173
#, python-format
msgid "Deleted old symlink %s."
msgstr ""

#: core/groups.py:1177
msgid "Unable to read symlink {0} (was: {1})."
msgstr ""

#: core/groups.py:1185
msgid "User {0} lacks the symlink to group shared dir {1}. Create it?"
msgstr ""

#: core/groups.py:1195
msgid "User {0} lacks the symlink to group shared dir {1}."
msgstr ""

#: core/groups.py:1220
msgid "Group {0} already in the list of profile {1}."
msgstr ""

#: core/groups.py:1256
msgid "Deleting group {0} from profile {1}."
msgstr ""

#: core/groups.py:1266
msgid "Group {0} already absent from profile {1}."
msgstr ""

#: core/groups.py:1321
msgid "Shared dir {0} does not exist, please run \"chk group {1}\" to fix."
msgstr ""

#: core/groups.py:1374
#, python-format
msgid ""
"Cannot demote user %s from standard membership to guest without --force flag."
msgstr ""

#: core/groups.py:1388
#, python-format
msgid "Cannot demote user %s from responsible to guest without --force flag."
msgstr ""

#: core/groups.py:1419
#, python-format
msgid ""
"Cannot demote user %s from responsible to standard membership without --"
"force flag."
msgstr ""

#: core/groups.py:1462
msgid "responsibles"
msgstr ""

#: core/groups.py:1465
msgid "guests"
msgstr ""

#: core/groups.py:1471
#, python-format
msgid "Checking system group %sâ¦"
msgstr ""

#: core/groups.py:1475
msgid ""
"The system group {0} is required for the group {1} to be fully operationnal. "
"Create it?"
msgstr ""

#: core/groups.py:1487
msgid "{0} of group â{1}â"
msgstr ""

#: core/groups.py:1513
msgid ""
"The system group {0} is required for the group {1} to be fully operationnal."
msgstr ""

#: core/groups.py:1535
msgid "Checking system specific attributes for group {0}â¦"
msgstr ""

#: core/groups.py:1546
msgid "Removed superfluous attribute {0} of system group {1}"
msgstr ""

#: core/groups.py:1556 core/groups.py:2246
#, python-format
msgid "Members of group â%sâ"
msgstr ""

#: core/groups.py:1558
msgid "Added missing {0} attribute with a default value for system group {1}."
msgstr ""

#: core/groups.py:1565
msgid "Do you want to commit these changes to the system (highly recommended)?"
msgstr ""

#: core/groups.py:1574
msgid "Corrections of system group {0} not commited"
msgstr ""

#: core/groups.py:1607
msgid "Unable to create symlink {0} (was: {1})."
msgstr ""

#: core/groups.py:1678
msgid "password:"
msgstr ""

#: core/groups.py:1682
msgid "skel:"
msgstr ""

#: core/groups.py:1691
msgid "UNKNOWN"
msgstr ""

#: core/groups.py:1693 core/groups.py:2339
msgid "permissive"
msgstr ""

#: core/groups.py:1695
msgid "NOT permissive"
msgstr ""

#: core/groups.py:1705 core/groups.py:1738 core/groups.py:1754
msgid "âÂ no memberÂ â"
msgstr ""

#: core/groups.py:1716
msgid "Cannot collect responsibles for group {0} (was: {1})."
msgstr ""

#: core/groups.py:1725
msgid "Cannot collect guests for group {0} (was: {1})."
msgstr ""

#: core/groups.py:1930
msgid "{0}.reload: Overwritten gid {1}."
msgstr ""

#: core/groups.py:1944
msgid "{0}: removing disapeared group {1}."
msgstr ""

#: core/groups.py:1987
msgid "File-system {0} must be mounted with {1} option to continue!"
msgstr ""

#: core/groups.py:2000
msgid "{0}: {1} does not exist and will be created later in the process."
msgstr ""

#: core/groups.py:2146
#, python-format
msgid "GID %s does not exist on the system."
msgstr ""

#: core/groups.py:2234
msgid "Malformed group name \"{0}\", must match /{1}/i."
msgstr ""

#: core/groups.py:2239
msgid ""
"Group name must be smaller than {0} characters (\"{1}\" is {2} chars long)."
msgstr ""

#: core/groups.py:2249
msgid "Malformed group description \"{0}\", must match /{1}/i."
msgstr ""

#: core/groups.py:2314
msgid "Created system group {0} (gid={1})."
msgstr ""

#: core/groups.py:2338
msgid "Created {0} group {1} (gid={2})."
msgstr ""

#: core/groups.py:2372
msgid "GID {0} is already taken by group {1}. Please choose another one."
msgstr ""

#: core/groups.py:2385 core/groups.py:2398
#, python-format
msgid "The group %s already exists."
msgstr ""

#: core/groups.py:2388
#, python-format
msgid ""
"The group %s already exists but has not the same type. Please choose another "
"name for your group."
msgstr ""

#: core/groups.py:2401
#, python-format
msgid ""
"The group %s already exists with a different GID. Please check this is what "
"you want, and make a decision."
msgstr ""

#: core/groups.py:2406
msgid ""
"The group {0} already exists but has not the same type. Please choose "
"another name for your group."
msgstr ""

#: core/groups.py:2422
#, python-format
msgid ""
"A user account called %s already exists, this could trigger a bug in the "
"Debian/Ubuntu adduser code when deleting the user. Please choose another "
"name for your group, or use the --force argument if you really want to add "
"this group on the system."
msgstr ""

#: core/groups.py:2440
msgid "Autogenerated GID for group {0}: {1}."
msgstr ""

#: core/groups.py:2447
msgid ""
"GID out of range for the kind of group you specified. System GIDs must be "
"between {0} and {1}, standard GIDs must be between {2} and {3}."
msgstr ""

#: core/groups.py:2508
msgid ""
"Cannot delete group {0}, currently primary group of profile {1}. Please "
"delete the profile instead, and the group will be deleted at the same time."
msgstr ""

#: core/groups.py:2519
msgid ""
"The group still has members. You must delete them first, or use the --del-"
"users argument. WARNING: this is usually a bad idea; use with caution."
msgstr ""

#: core/groups.py:2531
#, python-format
msgid ""
"Cannot delete a helper group. Please delete the standard associated group %s "
"instead, and this group will be deleted in the same time."
msgstr ""

#: core/groups.py:2538
#, python-format
msgid ""
"Cannot delete restricted system group %s without the --force argument, this "
"is too dangerous."
msgstr ""

#: core/groups.py:2646
msgid "{0}: group {1} already not referenced in controller!"
msgstr ""

#: core/groups.py:2671
msgid "Deleted {0}group {1}."
msgstr ""

#: core/groups.py:2672
msgid "system "
msgstr ""

#: core/groups.py:2749
#, python-format
msgid ""
"%s: inconsistency detected. Automatic check requested in the background."
msgstr ""

#: core/groups.py:2756
#, python-format
msgid ""
"%s: dangling helper group(s) detected. Automatic removal requested in the "
"background."
msgstr ""

#: core/groups.py:2802
msgid ""
"{0}: primary group is missing for user(s) {1}. This is harmless, but should "
"be corrected manually by re-creating groups with fixed GIDs, or changing "
"these users GIDs to other values."
msgstr ""

#: core/groups.py:2807
msgid "{0}(gid={1})"
msgstr ""

#: core/groups.py:2825
msgid "You must specify a GID or a name to test existence of."
msgstr ""

#: core/groups.py:2832
#, python-format
msgid "GID %s doesn't exist"
msgstr ""

#: core/groups.py:2841
#, python-format
msgid "Group %s doesn't exist"
msgstr ""

#: core/_controllers.py:87
msgid "Skipped non-existing {0} or {1} {2}."
msgstr ""

#: core/_controllers.py:97
msgid "\"{0}\" is neither an attribute nor an item of {1}."
msgstr ""

#: core/_controllers.py:489
msgid "{0} user rule that wanted to overwrite system default rule."
msgstr ""

#: core/_controllers.py:491 core/_modules.py:137 core/_modules.py:341
msgid "Skipped"
msgstr ""

#: core/_controllers.py:515
msgid ""
"{0} group rule for path {1} ({2}:{3}), overriden by system rule ({4}:{5})."
msgstr ""

#: core/_controllers.py:518
msgid "Ignored"
msgstr ""

#: core/_controllers.py:532
msgid "{0} {1} ACL rule: \"{2}\" for \"{3}\"."
msgstr ""

#: core/_controllers.py:534 core/_controllers.py:630
msgid "Added"
msgstr ""

#: core/_controllers.py:571
#, python-format
msgid "There is no default rule. Check %s."
msgstr ""

#: core/_controllers.py:572
msgid "aborted"
msgstr ""

#: core/_controllers.py:629
msgid "{0} {1} ACL rule {2}: \"{3}\" for \"{4}\"."
msgstr ""

#: core/_controllers.py:632
msgid "template"
msgstr ""

#: core/_controllers.py:637 core/_controllers.py:641
msgid "{0}.parse_rules: {1}"
msgstr ""

#: core/_controllers.py:667
msgid "parse_rules(): Exception on {0} (special={1})."
msgstr ""

#: core/_controllers.py:690
msgid "{0:s}: installing watchesâ¦"
msgstr ""

#: core/_controllers.py:699
msgid "{0}: exception while installing inotifier watches for {1} {2}"
msgstr ""

#: core/machines.py:46 core/machines.py:63
msgid "Failed to call `{0}()` on machine {1}!"
msgstr ""

#: core/machines.py:49 core/machines.py:66
msgid "Cannot call `{0}` on non-connected machine {1}!"
msgstr ""

#: core/machines.py:260
msgid "no other link than machine yet"
msgstr ""

#: core/machines.py:262
msgid "{0}: Linked {1} to {2}."
msgstr ""

#: core/machines.py:299
msgid ""
"{0}: guess_os({1}) â unknown key \"{2}\" with value \"{2}\", please update "
"our knowledge database if you what type of device it is."
msgstr ""

#: core/machines.py:306
msgid ""
"{0}: guess_os({1}) â unknown {2} value \"{3}\", please update our knowledge "
"database if you what type of device it is."
msgstr ""

#: core/machines.py:387
msgid "Could not resolve machine name for address {0}"
msgstr ""

#: core/machines.py:451
msgid "{0}: {1} already pyroized, not doing the work again."
msgstr ""

#: core/machines.py:526 core/machines.py:644 core/machines.py:665
msgid "{0}: {1} at {2}."
msgstr ""

#: core/machines.py:527
#, python-format
msgid "LicornÂ® %s shutdown"
msgstr ""

#: core/machines.py:527 core/machines.py:645 core/machines.py:667
msgid "server"
msgstr ""

#: core/machines.py:528 core/machines.py:646 core/machines.py:667
msgid "client"
msgstr ""

#: core/machines.py:555
msgid "{0}: _pyro_forward_goodbye_from(): harmless error {1} from {2}."
msgstr ""

#: core/machines.py:645
#, python-format
msgid "new LicornÂ® %s"
msgstr ""

#: core/machines.py:666
#, python-format
msgid "LicornÂ® %s back online"
msgstr ""

#: core/machines.py:673
msgid "{0}: {1} came back online."
msgstr ""

#: core/machines.py:695
msgid "Shut down machine {0}."
msgstr ""

#: core/machines.py:744
msgid "machine"
msgstr ""

#: core/machines.py:746
msgid "MID"
msgstr ""

#: core/machines.py:856
#, python-format
msgid "Overwritten mid %s"
msgstr ""

#: core/machines.py:887
msgid "{0:s}: cannot restart machine {1:s} (was: {2:s})"
msgstr ""

#: core/machines.py:956
msgid ""
"{0}: network auto-discovery disabled by configuration rule {1}, not going "
"further."
msgstr ""

#: core/machines.py:969
msgid "{0}: {1} initial network discovery."
msgstr ""

#: core/machines.py:970
msgid "starting"
msgstr ""

#: core/machines.py:1189
msgid "Package status unavailable on machine {0}."
msgstr ""

#: core/machines.py:1192
msgid ", {0}"
msgstr ""

#: core/machines.py:1193
msgid "packages status unavailable"
msgstr ""

#: core/machines.py:1195
msgid ", {0} package(s) to upgrade {1}"
msgstr ""

#: core/machines.py:1197
msgid "({0} security update(s))"
msgstr ""

#: core/machines.py:1206
msgid " (this machine)"
msgstr ""

#: core/machines.py:1208
msgid "{0}: {1}{2}{3}"
msgstr ""

#: core/machines.py:1209
msgid "status"
msgstr ""

#: core/machines.py:1211
msgid " (remote control enabled)"
msgstr ""

#: core/machines.py:1214
msgid "{0}: {1} ({2}{3})"
msgstr ""

#: core/machines.py:1215
msgid "address"
msgstr ""

#: core/machines.py:1217
#, python-format
msgid "expires: %s, "
msgstr ""

#: core/machines.py:1222
msgid "managed"
msgstr ""

#: core/machines.py:1223
msgid "floating"
msgstr ""

#: core/machines.py:1226
msgid "ethernet"
msgstr ""

#: core/machines.py:1306
#, python-format
msgid "MID %s does not exist"
msgstr ""

#: core/machines.py:1312
msgid "You must pass a hostname to verify!"
msgstr ""

#: core/machines.py:1319
msgid "Cannot build a valid hostname (got {0}, which does not verify {1})."
msgstr ""

#: core/_modules.py:55
msgid "module"
msgstr ""

#: core/_modules.py:58
msgid "module name"
msgstr ""

#: core/_modules.py:135
msgid "{0} unusable {1} {2}, exception at import"
msgstr ""

#: core/_modules.py:189
msgid "{0}: disabling dependant {1}(s) {2}."
msgstr ""

#: core/_modules.py:224
msgid ""
"{0}: will not try to load {1} {2} because one or more of its dependancies "
"failed to load."
msgstr ""

#: core/_modules.py:285
msgid "{0}: exception in {1} {2} during load"
msgstr ""

#: core/_modules.py:306
msgid "{0}: cannot disable non-existing {1} {2}."
msgstr ""

#: core/_modules.py:325
msgid ""
"{0} {1} is enabled on the server side but not available locally, probably an "
"installation problem."
msgstr ""

#: core/_modules.py:332
msgid ""
"{0} {1} is enabled on the server side but manually ignored locally in {2}, "
"please fix the problem before continuing."
msgstr ""

#: core/_modules.py:339
msgid "{0}: {1} {2} {3}, manually ignored in {4}."
msgstr ""

#: core/_modules.py:384 core/_modules.py:427
msgid "No {0} by that name \"{1}\"."
msgstr ""

#: core/_modules.py:388
msgid "{0} {1} already enabled."
msgstr ""

#: core/_modules.py:397
msgid "successfully enabled {0} {1}."
msgstr ""

#: core/_modules.py:402
msgid "{0} {1} would not enable itself."
msgstr ""

#: core/_modules.py:407
msgid "Exception while enabling {0} {1}."
msgstr ""

#: core/_modules.py:432
msgid "{0} {1} already disabled."
msgstr ""

#: core/_modules.py:442
msgid "Exception while unregistering events handlers/callbacks of {0} {1}"
msgstr ""

#: core/_modules.py:449
msgid "successfully disabled {0} {1}."
msgstr ""

#: core/_modules.py:454
msgid "{0} {1} would not disable itself."
msgstr ""

#: core/_modules.py:460
msgid "Exception while disabling {0} {1}."
msgstr ""

#: core/_modules.py:499
msgid "{0}: unhandled exception during {1} {2} check. Disabling it."
msgstr ""

#: core/_modules.py:522
msgid "{0}: global check finished."
msgstr ""

#: core/_modules.py:530
msgid "{0} {1} does not exist or is not enabled."
msgstr ""

#: core/_modules.py:544
msgid ""
"{name}: problem while installing inotifier watches for {type} {module} (was: "
"{exc})."
msgstr ""

#: core/_modules.py:665
msgid ""
"â³ {0} ({1} {2}):\n"
"\t{3}\n"
"{4}"
msgstr ""

#: core/_modules.py:668
msgid "client/server"
msgstr ""

#: core/_modules.py:670
msgid "depends on: {0}"
msgstr ""

#: core/_modules.py:671
msgid "no dependancies"
msgstr ""

#: core/keywords.py:96
#, python-format
msgid "%s: saving data structures to disk."
msgstr ""

#: core/keywords.py:107
#, python-format
msgid "%s: data structures saved to disk."
msgstr ""

#: core/keywords.py:141
msgid "You must specify a keyword name."
msgstr ""

#: core/keywords.py:145
#, python-format
msgid "A keyword named \"%s\" already exists!"
msgstr ""

#: core/keywords.py:149
msgid "The parent you specified does not exist."
msgstr ""

#: core/keywords.py:153
msgid "Malformed keyword name \"{0}\", must match /{1}/i."
msgstr ""

#: core/keywords.py:159
msgid "Malformed keyword description \"{0}\", must match /{1}/i."
msgstr ""

#: core/keywords.py:170
#, python-format
msgid "Added keyword %s."
msgstr ""

#: core/keywords.py:186
msgid ""
"The keyword is a parent which has children. If you want to delete children "
"too, use option --del-children."
msgstr ""

#: core/keywords.py:191
#, python-format
msgid "Deleted keyword %s."
msgstr ""

#: core/keywords.py:200 core/keywords.py:255 core/keywords.py:273
#: core/keywords.py:282
msgid "The keyword you specified does not exist."
msgstr ""

#: core/keywords.py:264
#, python-format
msgid "The keyword %s does not exist."
msgstr ""

#: core/keywords.py:314 core/keywords.py:362
msgid "Cannot get current keywords."
msgstr ""

#: core/keywords.py:326 core/keywords.py:378
msgid "Applyed {0} xattr {1} on {2}."
msgstr ""

#: core/keywords.py:332 core/keywords.py:384
msgid "Unable to modify {0} xattr of {1} (was: {2})."
msgstr ""

#: core/keywords.py:372
msgid "Removed xattr {0} from {1}."
msgstr ""

#: core/keywords.py:411
#, python-format
msgid "Removed xattr from %s."
msgstr ""

#: upgrades/common.py:38
msgid "Fixing permissions in {0}, this may take a whileâ¦"
msgstr ""

#: upgrades/core/backends/openldap/aaa_gecos_utf8.py:34
msgid "{0}: impossible to reload {1} schema into {2}!"
msgstr ""

#: upgrades/core/backends/openldap/aaa_gecos_utf8.py:38
msgid "{0}: updated {1} schema to support utf-8 gecos."
msgstr ""

#: upgrades/core/aaa_first_installation.py:23
msgid "configuration"
msgstr ""

#: upgrades/core/aaa_first_installation.py:24
msgid "cache"
msgstr ""

#: upgrades/core/aaa_first_installation.py:25
msgid "data"
msgstr ""

#: upgrades/core/aaa_first_installation.py:32
msgid "Created {1} directory {0}."
msgstr ""

#: upgrades/core/aaa_first_installation.py:36
msgid "Could not create {1} directory {0}! This is mandatory, exiting"
msgstr ""

#: upgrades/extensions/squid/aaa_rename_apt_proxy.py:24
msgid "{0}: could not rename {1} to {2}"
msgstr ""

#: upgrades/extensions/squid/aaa_rename_apt_proxy.py:29
msgid "{0}: renamed {1} to {2}."
msgstr ""

#: upgrades/__init__.py:36
msgid "Looking for upgrade modules in {0}â¦"
msgstr ""

#: upgrades/__init__.py:58
msgid "Successfully imported upgrade module {0}."
msgstr ""

#: upgrades/__init__.py:63
msgid "Upgrade module {0} is unusable."
msgstr ""

#: upgrades/__init__.py:71
msgid "Please fix it before continuing {0}"
msgstr ""

#: upgrades/__init__.py:73
msgid "(killing myself -9 NOW)"
msgstr ""

#: upgrades/daemon/aaa_wmi2_move_db.py:44
msgid ""
"Unrecoverable situation while trying to move the WMI db to its final and "
"proper location: either the old is already moved, or the Django settings are "
"not up-to-date; there could be another cause. Please fix the situation "
"manually"
msgstr ""

#: upgrades/daemon/aaa_wmi2_move_db.py:66
msgid "Could not delete {0}, continuing"
msgstr ""

#: upgrades/daemon/aaa_wmi2_move_db.py:86
msgid "Could not delete {0}, which should now have been empty."
msgstr ""

#: upgrades/daemon/aaa_wmi2_move_db.py:93
msgid ""
"Successfully moved Django WMI database to {0} and updated Django settings."
msgstr ""

#: upgrades/daemon/bbb_wmi2_setup.py:175
msgid "Creating the Django session database in {0}, this may take a whileâ¦"
msgstr ""

#: upgrades/daemon/bbb_wmi2_setup.py:190
msgid "Successfully created Django session database in {0}."
msgstr ""

#: upgrades/daemon/bbb_wmi2_setup.py:205
msgid "Creating an SSL certificate/key pair for the WMIâ¦"
msgstr ""

#: upgrades/daemon/bbb_wmi2_setup.py:234
msgid "Successfully created WMI SSL certificate/key pair."
msgstr ""

#: extensions/simplesharing.py:71
msgid "{0}: \"{1}\" must be a directory!"
msgstr ""

#: extensions/simplesharing.py:100
msgid "{0}: configuration file {1} seems corrupt; removing it."
msgstr ""

#: extensions/simplesharing.py:176
msgid "Please set a password on the share first."
msgstr ""

#: extensions/simplesharing.py:229
msgid "Bad URI {0}"
msgstr ""

#: extensions/simplesharing.py:297
msgid "Not a file from that share."
msgstr ""

#: extensions/simplesharing.py:301
msgid "Unsafe absolute path for file \"{0}\"!"
msgstr ""

#: extensions/simplesharing.py:311
msgid "Unknown"
msgstr ""

#: extensions/simplesharing.py:336
msgid "{0}: requesting short URLâ¦"
msgstr ""

#: extensions/simplesharing.py:343
msgid "{0}: short URL request failed, deferring next try."
msgstr ""

#: extensions/simplesharing.py:351
msgid "{0}: short URL successfully set to {1}."
msgstr ""

#: extensions/simplesharing.py:355
msgid ""
"{0}: short URL request failed, deferring next try (was: code={1}, message="
"{2})."
msgstr ""

#: extensions/simplesharing.py:363
msgid "{0}: {1} is not connected, deferring short URL request."
msgstr ""

#: extensions/simplesharing.py:411
msgid "Someone tried to switch simple shares on a non-standard user account!"
msgstr ""

#: extensions/simplesharing.py:415
msgid "{0}: shares status unchanged."
msgstr ""

#: extensions/simplesharing.py:426
msgid "{0}: simple shares status switched to {1}."
msgstr ""

#: extensions/simplesharing.py:449
msgid "{0}: checking sharesâ¦"
msgstr ""

#: extensions/simplesharing.py:456
msgid "{0}: shares check finished."
msgstr ""

#: extensions/simplesharing.py:460
msgid "{0}: no shares to check."
msgstr ""

#: extensions/simplesharing.py:464
msgid "User {0} home lacks the {1} directory to hold simple shares. Create it?"
msgstr ""

#: extensions/simplesharing.py:473
msgid "Created simple web shares directory {0}."
msgstr ""

#: extensions/simplesharing.py:478
msgid "User {0} does not accept simple shares, check skipped."
msgstr ""

#: extensions/simplesharing.py:502
msgid "{0}: error while listing shares, launching a check in the background."
msgstr ""

#: extensions/simplesharing.py:564 extensions/mylicorn/mylicorn.py:199
msgid "{0}: extension always enabled unless manually ignored in {1}."
msgstr ""

#: extensions/simplesharing.py:604
msgid "{0}: Exception while setting up shares for user {1}"
msgstr ""

#: extensions/simplesharing.py:613
msgid "{0}: checking all users' sharesâ¦"
msgstr ""

#: extensions/simplesharing.py:618
msgid "{0}: shares checks finished."
msgstr ""

#: extensions/powermanagement.py:107
msgid "{0}: extension enabled{1}, on top of {2} v{3}."
msgstr ""

#: extensions/powermanagement.py:109
msgid " with laptop mode"
msgstr ""

#: extensions/powermanagement.py:116
msgid ""
"{0}: extension disabled because either {1} disabled or {2} not connected."
msgstr ""

#: extensions/powermanagement.py:146
msgid "{0}: UPower message{1}: {2}"
msgstr ""

#: extensions/powermanagement.py:148
msgid " for device {0}"
msgstr ""

#: extensions/powermanagement.py:155
msgid "{0}: system is {1}, broadcasting information."
msgstr ""

#: extensions/powermanagement.py:157
msgid "going to sleep"
msgstr ""

#: extensions/powermanagement.py:159
msgid "resuming from suspend"
msgstr ""

#: extensions/openssh.py:87
msgid "{0}: not available because {1} or {2} do not exist on the system."
msgstr ""

#: extensions/openssh.py:119
msgid ""
"{0}: extension available on top of {1} version {2}, service currently {3}."
msgstr ""

#: extensions/openssh.py:156 extensions/samba3/samba3.py:122
msgid "{0}: group {1} must be created. Do it?"
msgstr ""

#: extensions/openssh.py:162
msgid "Users allowed to connect via SSHd"
msgstr ""

#: extensions/openssh.py:165 extensions/samba3/samba3.py:132
msgid "{0}: group {1} must exist before continuing."
msgstr ""

#: extensions/openssh.py:170
msgid "{0}: checking good default values in {1}â¦"
msgstr ""

#: extensions/openssh.py:180
msgid "{0}: {1} must be modified. Do it?"
msgstr ""

#: extensions/openssh.py:186
msgid "{0}: written configuration file {1}."
msgstr ""

#: extensions/openssh.py:190 foundations/classes.py:117
msgid "{0}: configuration file {1} must be altered to continue."
msgstr ""

#: extensions/gloop.py:113
msgid ""
"{0}: extension not available because {1} or {2} do not exist on the system."
msgstr ""

#: extensions/gloop.py:144
msgid "{0}: could not acquire {1} system bus, disabling"
msgstr ""

#: extensions/gloop.py:150
msgid "{0}: started extension and gobject mainloop thread."
msgstr ""

#: extensions/gloop.py:156
msgid "{0}: DBUS message {1} {2}."
msgstr ""

#: extensions/__init__.py:137
msgid "Started service {0} ({1})."
msgstr ""

#: extensions/__init__.py:138
msgid "Stopped service {0} ({1})."
msgstr ""

#: extensions/__init__.py:139
msgid "Restarted service {0} ({1})."
msgstr ""

#: extensions/__init__.py:140
msgid "Reloaded service {0} ({1})."
msgstr ""

#: extensions/__init__.py:146
#, python-format
msgid "Starting service %s. Please wait, this can take a while."
msgstr ""

#: extensions/__init__.py:147
#, python-format
msgid "Stopping service %s. Please wait, this can take a while."
msgstr ""

#: extensions/__init__.py:148
#, python-format
msgid "Restarting service %s. Please wait, this can take a while."
msgstr ""

#: extensions/__init__.py:149
#, python-format
msgid "Reloading service %s. Please wait, this can take a while."
msgstr ""

#: extensions/__init__.py:272
msgid "{0}: waiting for {1} command to complete before queuing {2}."
msgstr ""

#: extensions/__init__.py:281
#, python-format
msgid "%s: waiting 5 seconds more."
msgstr ""

#: extensions/__init__.py:289
msgid ""
"{0}: command {1} did not complete in {2} seconds, cannot exectute a "
"different one. Please try  again later."
msgstr ""

#: extensions/squid.py:214
msgid "{0}: not available because {1} do not exist on the system."
msgstr ""

#: extensions/squid.py:236 extensions/caldavd.py:151
msgid "{0}: started extension, managing {1} service ({2})."
msgstr ""

#: extensions/squid.py:240 extensions/caldavd.py:154
#, python-format
msgid "pid=%s"
msgstr ""

#: extensions/squid.py:243 extensions/caldavd.py:157
msgid "Starting up"
msgstr ""

#: extensions/mylicorn/datasrc/base.py:70
msgid ""
"Another genfunc is already registered for this anonymous data mapping: old="
"\"{0}\", new=\"{1}\". This will certainly produce errors on the remote data "
"collector side."
msgstr ""

#: extensions/mylicorn/datasrc/base.py:133 extensions/rdiffbackup.py:942
msgid "{0}: created directory {1}."
msgstr ""

#: extensions/mylicorn/datasrc/wtmp.py:79
msgid "{0}: skipping duplicate entry {1}; WTMP file {2} is probably corrupt."
msgstr ""

#: extensions/mylicorn/datasrc/wtmp.py:121
msgid ""
"{0}: skipped orphaned entry {1} with no start; either it was opened before "
"WTMP file {2} was rotated, either the file is corrupt."
msgstr ""

#: extensions/mylicorn/datasrc/wtmp.py:170
msgid ""
"{0}: There are {1} still-open entries in WTMP database. Not yielded, perhaps "
"next time if they get closed until then."
msgstr ""

#: extensions/mylicorn/mylicorn.py:113
msgid "Bad API key \"{0}\", must match \"{1}/i\""
msgstr ""

#: extensions/mylicorn/mylicorn.py:144
msgid "{0}: using environment variable {1} pointing to {2}."
msgstr ""

#: extensions/mylicorn/mylicorn.py:174
msgid "{0}: configuration file {1} seems corrupt; not using it."
msgstr ""

#: extensions/mylicorn/mylicorn.py:297
msgid "{0}: updater thread started."
msgstr ""

#: extensions/mylicorn/mylicorn.py:308
msgid "{0}: exception while stopping updater thread"
msgstr ""

#: extensions/mylicorn/mylicorn.py:317
msgid "{0}: updater thread stopped."
msgstr ""

#: extensions/mylicorn/mylicorn.py:329
msgid "{0}: error decoding web exception {1} from failed execution of {2}"
msgstr ""

#: extensions/mylicorn/mylicorn.py:333
msgid "{0}: error while executing {1}"
msgstr ""

#: extensions/mylicorn/mylicorn.py:337
#, python-format
msgid "RPC %s failed"
msgstr ""

#: extensions/mylicorn/mylicorn.py:360
msgid "{0}: reprogramming authentication in {1}."
msgstr ""

#: extensions/mylicorn/mylicorn.py:419
msgid "{0}: failed to disconnect (code: {1}, message: {2})"
msgstr ""

#: extensions/mylicorn/mylicorn.py:426
msgid "{0}: sucessfully disconnected (code: {1}, message: {2})"
msgstr ""

#: extensions/mylicorn/mylicorn.py:435
msgid "{0}: already disconnected, not trying again."
msgstr ""

#: extensions/mylicorn/mylicorn.py:443
msgid "{0}: already connected, not doing it again."
msgstr ""

#: extensions/mylicorn/mylicorn.py:448
msgid ""
"{0}: system UUID not found, aborting. There may be a serious problem "
"somewhere."
msgstr ""

#: extensions/mylicorn/mylicorn.py:462
msgid "{0}: failed to authenticate (code: {1}, message: {2})"
msgstr ""

#: extensions/mylicorn/mylicorn.py:468
msgid "{0}: sucessfully authenticated (code: {1}, message: {2})"
msgstr ""

#: extensions/mylicorn/mylicorn.py:495
msgid "{0}: our reachability state is now {1}."
msgstr ""

#: extensions/mylicorn/mylicorn.py:511
msgid "{0}: problem noop()'ing {1} (was: code={2}, message={3})."
msgstr ""

#: extensions/mylicorn/mylicorn.py:526
msgid "{0}: successfully noop()'ed {1}."
msgstr ""

#: extensions/mylicorn/mylicorn.py:551
msgid "{0}: uploading {1}{2} history data, starting from hash {3}â¦"
msgstr ""

#: extensions/mylicorn/mylicorn.py:553
msgid "fully anonymized "
msgstr ""

#: extensions/mylicorn/mylicorn.py:555
msgid "anonymized "
msgstr ""

#: extensions/mylicorn/mylicorn.py:570
msgid "{0}: problem uploading {1} history data (was: code={2}, message={3})."
msgstr ""

#: extensions/mylicorn/mylicorn.py:578
msgid "{0}: successfully uploaded / updated {1} history data at {2}."
msgstr ""

#: extensions/mylicorn/mylicorn.py:584
msgid "{0}: no new data to send for {1} history since last upload."
msgstr ""

#: extensions/mylicorn/mylicorn.py:596
msgid ""
"{0}: problem querying for last {1} history value, skipping to next (was: "
"code={2}, message={3})."
msgstr ""

#: extensions/volumes.py:169
msgid "{0}: unhandled action {1} for device {2} ({3})."
msgstr ""

#: extensions/volumes.py:178
msgid "{0}: unknown action {1} for device {2} ({3})."
msgstr ""

#: extensions/volumes.py:269
msgid "volume {0}[{1}]{2}"
msgstr ""

#: extensions/volumes.py:272
#, python-format
msgid " (on %s)"
msgstr ""

#: extensions/volumes.py:273
msgid " (not mounted)"
msgstr ""

#: extensions/volumes.py:290
msgid "{0} {1}"
msgstr ""

#: extensions/volumes.py:322
msgid "{0}: LicornÂ® usage already enabled on {1}."
msgstr ""

#: extensions/volumes.py:330
msgid "{0}: enabled LicornÂ® usage on {1}."
msgstr ""

#: extensions/volumes.py:336
msgid "{0}: cannot enable LicornÂ® usage on {1} (unsupported FS {2})."
msgstr ""

#: extensions/volumes.py:361
msgid "{0}: disabled LicornÂ® usage on {1}."
msgstr ""

#: extensions/volumes.py:365
msgid "{0}: LicornÂ® usage already disabled on {1}."
msgstr ""

#: extensions/volumes.py:408
msgid "{0}: created mount point {1}."
msgstr ""

#: extensions/volumes.py:460
msgid "{0}: mounted device {1}({2}) at {3}."
msgstr ""

#: extensions/volumes.py:481
msgid "{0}: still got workers: {1}."
msgstr ""

#: extensions/volumes.py:484
msgid "Cannot unmount {0}, still busy in LicornÂ®."
msgstr ""

#: extensions/volumes.py:508
msgid "{0}: unmounted device {1} from {2}."
msgstr ""

#: extensions/volumes.py:519
msgid "{0}({1}): mount point {2} still busy after unmounting."
msgstr ""

#: extensions/volumes.py:526
msgid "{0}({1}): cannot remove mount point {2}: {3}."
msgstr ""

#: extensions/volumes.py:534
msgid "{0}: removed directory {1}."
msgstr ""

#: extensions/volumes.py:569
msgid "volume"
msgstr ""

#: extensions/volumes.py:572
msgid "device"
msgstr ""

#: extensions/volumes.py:597
msgid "No Volume by that device name, label, mount_point nor GUID \"{0}\"!"
msgstr ""

#: extensions/volumes.py:708
msgid "{0}: not available because"
msgstr ""

#: extensions/volumes.py:719
msgid "{0}: started extension with pyudev v{2} on top of udev v{1}."
msgstr ""

#: extensions/volumes.py:956
msgid ""
"{0}: computing global system size is not implemented for systems other than "
"Linux. Returning 0 and hoping this will be sufficient."
msgstr ""

#: extensions/volumes.py:980
msgid "{0}: skipped already known volume {1}."
msgstr ""

#: extensions/volumes.py:991
msgid "{0}: skipped LMV/extended partition {1}."
msgstr ""

#: extensions/volumes.py:997
msgid "{0}: skipped unformatted partition {1}."
msgstr ""

#: extensions/volumes.py:1003
msgid "{0}: skipped system partition {1}."
msgstr ""

#: extensions/volumes.py:1023
msgid "{0}: skipped partition {1} (excluded {2} filesystem)."
msgstr ""

#: extensions/volumes.py:1047
msgid "{0}: added {1}."
msgstr ""

#: extensions/volumes.py:1092
msgid "{0}: removed {1}."
msgstr ""

#: extensions/volumes.py:1136
msgid "{0}: skipped non existing device or mount_point {1}."
msgstr ""

#: extensions/volumes.py:1208
msgid ", {0} of {1} free ({2:.1%})"
msgstr ""

#: extensions/volumes.py:1216
msgid " on {0}{1}"
msgstr ""

#: extensions/volumes.py:1219
msgid " not mounted"
msgstr ""

#: extensions/samba3/netlogon.py:95
msgid "LicornÂ® netlogon script"
msgstr ""

#: extensions/samba3/netlogon.py:96
msgid "WARNING: do not modify this script directly:"
msgstr ""

#: extensions/samba3/netlogon.py:97
msgid ""
"it has been auto-generated, and will be overwritten next time the user logs "
"in."
msgstr ""

#: extensions/samba3/netlogon.py:98
msgid "If you would like to customize it, use templates from {0}"
msgstr ""

#: extensions/samba3/netlogon.py:99
msgid "for inspiration, and create new scripts in {0}."
msgstr ""

#: extensions/samba3/netlogon.py:100
msgid "For more information, visit http://docs.licorn.org/extensions/samba3"
msgstr ""

#: extensions/samba3/netlogon.py:101
msgid "Hello {0} and welcome to {1}!"
msgstr ""

#: extensions/samba3/netlogon.py:102
msgid "{0} generated on {1}"
msgstr ""

#: extensions/samba3/netlogon.py:193
msgid "{0}: WindowsÂ® profile rewrite for user {1} based on {2}â¦"
msgstr ""

#: extensions/samba3/netlogon.py:215
msgid ""
"{0}: exception while removing {1}, this could lead to inconsistencies in "
"{2}'s WindowsÂ® profile."
msgstr ""

#: extensions/samba3/netlogon.py:227
msgid ""
"{0}: exception while copying {1} to {2}, this could lead to inconsistencies "
"in {3}'s WindowsÂ® profile."
msgstr ""

#: extensions/samba3/netlogon.py:240
msgid "{0}: user {1} WindowsÂ® profile overwritten by {2}."
msgstr ""

#: extensions/samba3/netlogon.py:258
msgid ""
"{0}: exception while removing old script {1}, this will probably lead to "
"inconsistencies in {2}'s Windows session."
msgstr ""

#: extensions/samba3/netlogon.py:265
msgid "{0}: creating netlogon script for user {1} with levels {2}â¦"
msgstr ""

#: extensions/samba3/netlogon.py:292
msgid "{0}: user {1}, adding {2}."
msgstr ""

#: extensions/samba3/netlogon.py:308
msgid "{0}: written {1}' netlogon script {2}."
msgstr ""

#: extensions/samba3/samba3.py:91 extensions/rdiffbackup.py:365
msgid "{0}: extension enabled on top of {1} version {2}."
msgstr ""

#: extensions/samba3/samba3.py:100
msgid "{0}: extension disabled because {1}  nor ({2} and {3}) not found."
msgstr ""

#: extensions/samba3/samba3.py:115
msgid "WindowsÂ®(-only) domain administrators"
msgstr ""

#: extensions/samba3/samba3.py:116
msgid "WindowsÂ® workstations members of the MS domain"
msgstr ""

#: extensions/samba3/samba3.py:117
msgid "Users responsible of at least one group"
msgstr ""

#: extensions/samba3/samba3.py:288
msgid "{0}: Exception in user_post_add({1})"
msgstr ""

#: extensions/samba3/samba3.py:300
msgid "{0}: created {1}'s WindowsÂ® profile (empty) in {2}."
msgstr ""

#: extensions/samba3/samba3.py:334
msgid "{0}: Exception in user_post_change_password({1})"
msgstr ""

#: extensions/samba3/samba3.py:364
msgid "{0}: Exception in user_pre_del({1})"
msgstr ""

#: extensions/samba3/samba3.py:408
msgid "{0}: Error while adding user {1} to group {2}"
msgstr ""

#: extensions/samba3/samba3.py:442
msgid "{0}: Error while removing user {1} from group {2}"
msgstr ""

#: extensions/caldavd.py:83
msgid "{0}: extension not available because {1}."
msgstr ""

#: extensions/caldavd.py:88
msgid ""
"{0}: extension not yet available because {1} (calendarserver is probably "
"installing, try again later)."
msgstr ""

#: extensions/caldavd.py:94
msgid "{0}: extension not available because {1} not installed."
msgstr ""

#: extensions/caldavd.py:111
msgid ""
"**Please implement caldavd extension check for pre-existing users and groups."
"**"
msgstr ""

#: extensions/caldavd.py:163
msgid ""
"{0}: extension disabled because service {1} disabled on the local system."
msgstr ""

#: extensions/caldavd.py:368
msgid "{0}: unable to modify {1} for {2} {3}, not found in {4}."
msgstr ""

#: extensions/caldavd.py:383
msgid "{0}: unable to delete {1} {2}, not found in {3}."
msgstr ""

#: extensions/caldavd.py:401
msgid "{0}: unable to delete user {1} from group {2}, not found in {3}."
msgstr ""

#: extensions/caldavd.py:444
#, python-format
msgid "R/O holder for user %s."
msgstr ""

#: extensions/caldavd.py:699
msgid "Personnal calendar URI"
msgstr ""

#: extensions/caldavd.py:713
msgid "Group calendar URI"
msgstr ""

#: extensions/rdiffbackup.py:45
msgid "{0}: No default volume found, aborting call of {1}({2}, {3})"
msgstr ""

#: extensions/rdiffbackup.py:76
msgid "{0}: an operation is already in progress on {1}, aborting."
msgstr ""

#: extensions/rdiffbackup.py:269
msgid ""
"{0}: using environment variable {1} pointing to {2} for rdiff-backup "
"configuration."
msgstr ""

#: extensions/rdiffbackup.py:386
msgid "{0}: backup task scheduled."
msgstr ""

#: extensions/rdiffbackup.py:403
msgid "{0}: backup task un-scheduled."
msgstr ""

#: extensions/rdiffbackup.py:427
msgid ""
"{0}: cannot disable now, a backup is in progress. Please wait until the end "
"and retry."
msgstr ""

#: extensions/rdiffbackup.py:505
msgid "Last Backup"
msgstr ""

#: extensions/rdiffbackup.py:506
msgid "Increments listing"
msgstr ""

#: extensions/rdiffbackup.py:507
msgid "Statistics computation"
msgstr ""

#: extensions/rdiffbackup.py:533
msgid "{0}: no volume found!"
msgstr ""

#: extensions/rdiffbackup.py:589
#, python-format
msgid "Skipped disabled %s."
msgstr ""

#: extensions/rdiffbackup.py:649
msgid "{0}: Computing needed space for next backup, please waitâ¦"
msgstr ""

#: extensions/rdiffbackup.py:776
msgid "{0}: cleaning obsolete backup on {1}."
msgstr ""

#: extensions/rdiffbackup.py:781
msgid "{0}: obsolete backup cleaned on {1}."
msgstr ""

#: extensions/rdiffbackup.py:840
msgid ""
"Cannot run a backup on non-reserved volume {0}. Please reserve it first, or "
"choose another one."
msgstr ""

#: extensions/rdiffbackup.py:848
msgid "{0}: minimum backup interval cannot be less than {1}, clipping."
msgstr ""

#: extensions/rdiffbackup.py:856
msgid "{0}: not backing up on {1}, last backup is less than {2}."
msgstr ""

#: extensions/rdiffbackup.py:866
msgid "Sorry, no backup volumes available."
msgstr ""

#: extensions/rdiffbackup.py:934
msgid "{0}: running backup on {1}, please waitâ¦"
msgstr ""

#: extensions/rdiffbackup.py:962
msgid "{0}: configuration file {1} does not exist."
msgstr ""

#: extensions/rdiffbackup.py:1002
msgid "{0}: backup procedure terminated on {1}."
msgstr ""

#: extensions/rdiffbackup.py:1012
msgid ""
"Forcing backup on volume {0}, free space ckeck ignored. This could lead to "
"incomplete backups, please use with care!"
msgstr ""

#: extensions/rdiffbackup.py:1019
msgid "{0}: computing needed space for next backup, please wait."
msgstr ""

#: extensions/rdiffbackup.py:1025
msgid ""
"Volume {0} does not have enough free space or is not big enough to hold more "
"than two backups. Space needed: {1}; free space: {2}."
msgstr ""

#: extensions/rdiffbackup.py:1058
msgid "{0}: updated last backup file {1}."
msgstr ""

#: extensions/rdiffbackup.py:1072
msgid "{0}: computing statistics on {1}, please waitâ¦"
msgstr ""

#: extensions/rdiffbackup.py:1126
msgid "{0}: statistics computation finished on {1}."
msgstr ""

#: extensions/rdiffbackup.py:1202
msgid "No need to schedule the autobackup task yet."
msgstr ""

#: extensions/rdiffbackup.py:1214
msgid "Removing the auto-backup task is not allowed."
msgstr ""

#: daemon/inotifier.py:80
msgid "{0}{1} ({2} watched dirs, {3} config files, {4} queued events)"
msgstr ""

#: daemon/inotifier.py:108
msgid "{0}: closing {1} watches, please waitâ¦"
msgstr ""

#: daemon/inotifier.py:137
msgid "{0}: error on read_events: {1}"
msgstr ""

#: daemon/inotifier.py:161
msgid "inotifier: error deleting {0} (was: {1} {2})"
msgstr ""

#: daemon/inotifier.py:323
msgid "{0}: configuration file {1} changed, trigerring upstream reload."
msgstr ""

#: daemon/inotifier.py:355
msgid "{0}: increased inotify {1} to {2}."
msgstr ""

#: daemon/inotifier.py:365
msgid "{0}: unable to increase inotify {1} to {2}"
msgstr ""

#: daemon/inotifier.py:376
msgid "{0:s}: exception install watches of {1:s}."
msgstr ""

#: daemon/base.py:92
msgid "Console-initiated garbage collection and dead thread cleaning."
msgstr ""

#: daemon/base.py:97
msgid ""
"operate a manual garbage collection in the daemon's memory, and run other "
"clean-related things"
msgstr ""

#: daemon/base.py:106
msgid "show the INotifier watches (WARNING: this can flood your terminal)"
msgstr ""

#: daemon/base.py:109
msgid "{0} active threads: {1}"
msgstr ""

#: daemon/base.py:112
msgid "show all threads names and their current status"
msgstr ""

#: daemon/base.py:155
msgid "{0}: Send URS1 signal (and consequently reload daemon)"
msgstr ""

#: daemon/base.py:161
msgid "{0}: dump daemon status"
msgstr ""

#: daemon/base.py:167
msgid "{0} or {1}: dump daemon status after having cleared the screen"
msgstr ""

#: daemon/base.py:170
msgid "Space"
msgstr ""

#: daemon/base.py:182
msgid "{0}({1})"
msgstr ""

#: daemon/base.py:279
msgid "Problem getting interface {0} address (was: {1})."
msgstr ""

#: daemon/base.py:291
#, python-format
msgid "Bad IP address or hostname %s. Please check syntax."
msgstr ""

#: daemon/base.py:343
msgid "{0}: execvp({1})"
msgstr ""

#: daemon/base.py:351
msgid "transient process {0}: cannot execvp() {1}."
msgstr ""

#: daemon/base.py:445
msgid "{0}: skipped process @{1} which is not in the same cgroup."
msgstr ""

#: daemon/base.py:464
msgid "{0}: killed aborted instance @pid {1}."
msgstr ""

#: daemon/base.py:508
msgid ""
"Don't fork as a daemon, stay on the current terminal instead. Logs will be "
"printed on standard output instead of beiing written into the logfile."
msgstr ""

#: daemon/base.py:514
msgid ""
"Replace an existing daemon instance. A comfort flag to avoidkilling an "
"existing daemon before relaunching a new one."
msgstr ""

#: daemon/base.py:519
msgid ""
"Shutdown any currently running LicornÂ® daemon. We will try to terminate them "
"nicely, before beiing more agressive after a given period of time."
msgstr ""

#: daemon/base.py:525
msgid ""
"Specify an IP address or a hostname to bind to. Only {0} can be specified "
"(the WMI cannot yet bind on multiple interfaces at the same time). This "
"option takes precedence over the configuration directive, if present in {1}."
msgstr ""

#: daemon/base.py:534
#, python-format
msgid "Don't fork the WMI. This flag overrides the setting in %s."
msgstr ""

#: daemon/base.py:547
msgid ""
"Check the system for compliance with LicornÂ®. Meant to be used during first "
"installation. The daemon will check, auto-repair and install everything "
"necessary, and give the hand back. {0}; no need to supply \"-D\". Do not "
"misunderstand that the daemon will {1} run this check in normal conditions; "
"but with this flag it will {2} run the checks and exit immediately after "
"them."
msgstr ""

#: daemon/base.py:554
msgid "This option implies that the daemon will not fork into the background"
msgstr ""

#: daemon/base.py:556
msgid "always"
msgstr ""

#: daemon/base.py:557
msgid "only"
msgstr ""

#: daemon/base.py:567
msgid "{0}: running initial checks in the foreground."
msgstr ""

#: daemon/base.py:634
msgid "{0}: trying to {1} existing instance @pid {2}."
msgstr ""

#: daemon/base.py:635
msgid "replace"
msgstr ""

#: daemon/base.py:636
msgid "shutdown"
msgstr ""

#: daemon/base.py:647
msgid ""
"{0}: existing instance still running, we're going to be more incisive in a "
"few seconds."
msgstr ""

#: daemon/base.py:670
msgid ""
"{0}: re-killing old instance softly with TERM signal and waiting a little "
"more."
msgstr ""

#: daemon/base.py:687
msgid ""
"{0}: old instance has not terminated after 8 seconds. Sending KILL signal."
msgstr ""

#: daemon/base.py:715
#, python-format
msgid "{0}: killing old instance's father (sudo, pid %s) without any mercy."
msgstr ""

#: daemon/base.py:732
msgid ""
"{0}: old instance has not terminated after 9 seconds and cannot be killed. "
"We will not try to kill any other parent than \"{1}\", you are in a non-"
"trivial situation. Up to you to solve it."
msgstr ""

#: daemon/base.py:743
msgid ""
"{0}: old instance has not terminated after 15 seconds and cannot be killed "
"directly or by killing its direct parent, bailing out. You are in trouble on "
"a system where \"kill -9\" does not work as advertised. Sorry for you."
msgstr ""

#: daemon/base.py:753
msgid "{0}: old instance {1} terminated{2}"
msgstr ""

#: daemon/base.py:754
msgid "nastily"
msgstr ""

#: daemon/base.py:754
msgid "successfully"
msgstr ""

#: daemon/base.py:755
msgid ", we can play now."
msgstr ""

#: daemon/base.py:759
msgid "{0}: daemon already running (pid {1}), not restarting."
msgstr ""

#: daemon/base.py:773
msgid "{0}: must be run as {1} (was: {2})."
msgstr ""

#: daemon/base.py:781
msgid "{0}: unlinking PID file {1}."
msgstr ""

#: daemon/base.py:787
msgid "{0}: cannot unlink {1}."
msgstr ""

#: daemon/base.py:793
msgid " (up {0})"
msgstr ""

#: daemon/base.py:800
msgid "{0}: already stopping, ignored another terminate() call."
msgstr ""

#: daemon/base.py:803
msgid "{0}: already stopping, ignored signal {1}."
msgstr ""

#: daemon/base.py:812
msgid "{0}: cleaning up and stopping threadsâ¦"
msgstr ""

#: daemon/base.py:815
msgid "{0}: signal {1} received, shutting downâ¦"
msgstr ""

#: daemon/base.py:829
msgid "{0}: remaining threads before exit: {1}."
msgstr ""

#: daemon/base.py:832
msgid "{0}: exiting{1}."
msgstr ""

#: daemon/base.py:848
msgid "{0}: SIGUSR1 received, preparing our restart."
msgstr ""

#: daemon/base.py:851
msgid "{0}: restart needed, shutting everything down."
msgstr ""

#: daemon/main.py:125
msgid "{0}: initializing facilities, backends, controllers and extensions."
msgstr ""

#: daemon/main.py:142
msgid "{0}: {1} callbacks collected."
msgstr ""

#: daemon/main.py:270
msgid "{0}: collecting all threads."
msgstr ""

#: daemon/main.py:283
msgid "{0}: starting all threads."
msgstr ""

#: daemon/main.py:296
msgid "Could not start thread {0}"
msgstr ""

#: daemon/main.py:300
msgid "{0}: stopping threads."
msgstr ""

#: daemon/main.py:315
msgid ""
"{0}: thread {1} has no stop() nor cancel() method! It is probably still "
"running."
msgstr ""

#: daemon/main.py:354
msgid ""
"{0}: not starting WMI, disabled on command line or by configuration "
"directive."
msgstr ""

#: daemon/main.py:366
msgid "{0}: not starting the WMI because initial check in progress."
msgstr ""

#: daemon/main.py:388
msgid "{0}: initial check finished, terminating."
msgstr ""

#: daemon/main.py:447
msgid "{0}: LicornÂ® daemon version {1} role {2} warming upâ¦"
msgstr ""

#: daemon/main.py:479
msgid "{0}: all threads started, going to sleep waiting for signals."
msgstr ""

#: daemon/main.py:487
msgid "{0}: all threads started, waiting for the end of initial check."
msgstr ""

#: daemon/main.py:492
msgid "{0}: all threads started, ready for TTY interaction."
msgstr ""

#: daemon/main.py:590
msgid ""
"LicornÂ® {role} daemon {full}status: up {uptime}, {nb_threads} threads, "
"{nb_controllers} controllers, {nb_queues} queues, {nb_locked}/{nb_locks} "
"Mlocks, {sub_locked}/{sub_locks} Ulocks\n"
"CPU: usr {ru_utime:.3}s, sys {ru_stime:.3}s MEM: res {mem_res:.2}Mb shr "
"{mem_shr:.2}Mb ush {mem_ush:.2}Mb stk {mem_stk:.2}Mb\n"
msgstr ""

#: daemon/main.py:616
#, python-format
msgid "Mlocks:  %s\n"
msgstr ""

#: daemon/main.py:620
#, python-format
msgid "Ulocks:  %s\n"
msgstr ""

#: daemon/main.py:623
#, python-format
msgid "Threads: %s\n"
msgstr ""

#: daemon/main.py:628
#, python-format
msgid "Queues:  %s\n"
msgstr ""

#: daemon/main.py:734
msgid "Could not load gettext translations for language {0}"
msgstr ""

#: daemon/main.py:738
msgid "Successfully loaded gettext translations for language {0}."
msgstr ""

#: daemon/main.py:776
msgid "{0}: cannot announce shutdown to remote hosts (was: {1})."
msgstr ""

#: daemon/main.py:781
msgid "{0}: joining threads."
msgstr ""

#: daemon/main.py:795
msgid "{0}: waiting for thread {1} to finish{2}."
msgstr ""

#: daemon/main.py:798
#, python-format
msgid " (currently working on %s)"
msgstr ""

#: daemon/main.py:839
msgid "{0}: restarting{1}."
msgstr ""

#: daemon/main.py:873 daemon/main.py:880
msgid "{0}: wiped dead thread {1} from memory."
msgstr ""

#: daemon/main.py:883
msgid "{0}: doing manual garbage collection on {1}."
msgstr ""

#: daemon/threads.py:203
msgid "{0}: Exception in self.pre_run_method(), aborting."
msgstr ""

#: daemon/threads.py:213
msgid "{0}: Exception in self.run_action_method(), continuing."
msgstr ""

#: daemon/threads.py:221
msgid "{0}: Exception in self.post_run_method()"
msgstr ""

#: daemon/threads.py:263
msgid ""
"{0}{1} for {2} [{3} workers; wake up in {4}]\n"
"{5}"
msgstr ""

#: daemon/threads.py:308
msgid "{0}: maximum workers already  running, but qsize={1}."
msgstr ""

#: daemon/threads.py:517
msgid "{0}: timer reset after {1} elapsed."
msgstr ""

#: daemon/threads.py:650
msgid "Class method {0}.setup() has not been called!"
msgstr ""

#: daemon/threads.py:778
msgid "{0}: stopped during sleep, job {1} will not be run at all."
msgstr ""

#: daemon/threads.py:789
msgid "{0}: Exception encountered while running {1}"
msgstr ""

#: daemon/threads.py:807
#, python-format
msgid "%s: queue is now empty, going asleep waiting for jobs."
msgstr ""

#: daemon/threads.py:847
msgid "{0}: the \"target\" first (or named) argument is mandatory!"
msgstr ""

#: daemon/threads.py:858
msgid "{0}: the \"trigger_event\" second (or named) argument is mandatory!"
msgstr ""

#: daemon/wmi.py:133
#, python-format
msgid "{0}: socket already in use. waiting (total: %ds)."
msgstr ""

#: daemon/wmi.py:139
msgid "{0}: socket error {1}."
msgstr ""

#: daemon/wmi.py:144
msgid "{0}: {1} to answer requests at address {2}."
msgstr ""

#: daemon/wmi.py:145 daemon/cmdlistener.py:478
msgid "ready"
msgstr ""

#: daemon/rwi.py:206 daemon/rwi.py:234
msgid "Skipped non existing or invalid {0} or {1} \"{2}\"."
msgstr ""

#: daemon/rwi.py:249
#, python-format
msgid "You must specify at least one %s!"
msgstr ""

#: daemon/rwi.py:285
msgid "Harmless Exception encountered in RWI.get_daemon_status()"
msgstr ""

#: daemon/rwi.py:538
msgid "{0} distinct event(s), {1} handler(s) and {2} callback(s)"
msgstr ""

#: daemon/rwi.py:547
msgid ""
"Event: {0}\n"
"\tHandlers:{1}{2}\n"
"\tCallbacks:{3}{4}\n"
msgstr ""

#: daemon/rwi.py:551 daemon/rwi.py:556
msgid "{0} in module {1}"
msgstr ""

#: daemon/rwi.py:563
msgid "{0}: {1} handler(s), {2} callback(s).\n"
msgstr ""

#: daemon/rwi.py:660
msgid "You must specify an import filename."
msgstr ""

#: daemon/rwi.py:665
msgid "You must specify a profile."
msgstr ""

#: daemon/rwi.py:671
msgid "You must specify a firstname column number."
msgstr ""

#: daemon/rwi.py:677
msgid "You must specify a lastname column number."
msgstr ""

#: daemon/rwi.py:683
msgid "You must specify a group column number."
msgstr ""

#: daemon/rwi.py:690
msgid ""
"Two columns have the same number (lastname={0}, firstname={1}, group={2})"
msgstr ""

#: daemon/rwi.py:699
msgid "Sorry, CSV file must have no more than 128 columns.\""
msgstr ""

#: daemon/rwi.py:712
msgid ""
"Cannot automatically detect the file encoding, assuming iso-8859-15 "
"(latin-1)!"
msgstr ""

#: daemon/rwi.py:720
#, python-format
msgid "The profile \"%s\" does not exist."
msgstr ""

#: daemon/rwi.py:740
#, python-format
msgid "Cannot load CSV file (was: %s)."
msgstr ""

#: daemon/rwi.py:747
msgid "Reading input file: "
msgstr ""

#: daemon/rwi.py:782
msgid ""
"Import error on line {0}: no {1} specified or bad separator used(was: {2})."
msgstr ""

#: daemon/rwi.py:792
#, python-format
msgid "Encoding not supported for input filename (was: %s)."
msgstr ""

#: daemon/rwi.py:806 daemon/rwi.py:828
msgid ""
"Import error on line {0}: no group specified or bad {2} data (was: {1})."
msgstr ""

#: daemon/rwi.py:808 daemon/rwi.py:830
msgid "firstname or lastname"
msgstr ""

#: daemon/rwi.py:809 daemon/rwi.py:831 daemon/rwi.py:928 daemon/rwi.py:941
msgid "login"
msgstr ""

#: daemon/rwi.py:816 daemon/rwi.py:838
msgid "Import error on line {0} (was: {1})."
msgstr ""

#: daemon/rwi.py:861
msgid " done."
msgstr ""

#: daemon/rwi.py:894
msgid "Added group {0} ({1}/{2}); progress: {3}%"
msgstr ""

#: daemon/rwi.py:921 daemon/rwi.py:938
msgid "Fields order:"
msgstr ""

#: daemon/rwi.py:922
msgid "<table>"
msgstr ""

#: daemon/rwi.py:923 daemon/rwi.py:988
msgid "<tr>"
msgstr ""

#: daemon/rwi.py:926 daemon/rwi.py:939
msgid "FIRSTname"
msgstr ""

#: daemon/rwi.py:927 daemon/rwi.py:940
msgid "LASTname"
msgstr ""

#: daemon/rwi.py:932
msgid "<th width=\"{0}\">{1}</th>"
msgstr ""

#: daemon/rwi.py:933 daemon/rwi.py:999
msgid "</tr>"
msgstr ""

#: daemon/rwi.py:961
msgid ""
"Added user {0} {1} [login={2}, uid={3}, passwd={4}] ({5}/{6}); progress: {7}%"
msgstr ""

#: daemon/rwi.py:984
msgid ""
"Import error on line {0}.\n"
"{1}"
msgstr ""

#: daemon/rwi.py:995 daemon/rwi.py:1008
msgid "(autogenerated upon creation)"
msgstr ""

#: daemon/rwi.py:998
msgid "<td>{0}</td>"
msgstr ""

#: daemon/rwi.py:1033
msgid "Finished importing, creating summary HTML file: "
msgstr ""

#: daemon/rwi.py:1039 foundations/fsapi.py:1500
msgid "%d %m %Y at %H:%M:%S"
msgstr ""

#: daemon/rwi.py:1051
msgid ""
"This repport is available on the server at \"<code>{0}</code>\", you can <a "
"href=\"/system/download/{0}\">download it</a> for easier access"
msgstr ""

#: daemon/rwi.py:1081
msgid "{0} accounts and passwords"
msgstr ""

#: daemon/rwi.py:1082
msgid "Import made on {0}."
msgstr ""

#: daemon/rwi.py:1083
msgid ""
"Keeping passwords in any written form is a major security flaw for you "
"information system.<br />Please be sure obliterate this file and any printed "
"version after having transmitted their password to your users."
msgstr ""

#: daemon/rwi.py:1089
msgid "Direct access to {0}:"
msgstr ""

#: daemon/rwi.py:1107
msgid "Lastname"
msgstr ""

#: daemon/rwi.py:1108
msgid "Firstname"
msgstr ""

#: daemon/rwi.py:1109
msgid "Login"
msgstr ""

#: daemon/rwi.py:1110
msgid "Password"
msgstr ""

#: daemon/rwi.py:1141
msgid "done."
msgstr ""

#: daemon/rwi.py:1141
msgid "report:"
msgstr ""

#: daemon/rwi.py:1246
msgid "Unable to add user(s) {0} in group {1} (was: {2})."
msgstr ""

#: daemon/rwi.py:1446
msgid "Unable to resolve machine: {0}"
msgstr ""

#: daemon/rwi.py:1450
msgid "Error unpacking task args {0}"
msgstr ""

#: daemon/rwi.py:1465
msgid "Error unpacking task kwarg {0} from {1} (was: {2})"
msgstr ""

#: daemon/rwi.py:1514
msgid ""
"Bad event name \"{0}\". Use `get events` to display a list of valid event "
"names."
msgstr ""

#: daemon/rwi.py:1536
msgid "Bad priority \"{0}\""
msgstr ""

#: daemon/rwi.py:1568
msgid "No volume matched \"{0}\". Volume attributes are case sensitive."
msgstr ""

#: daemon/rwi.py:1579
msgid "Exception happened while launching backup on volume {0}"
msgstr ""

#: daemon/rwi.py:1605
msgid "Are you sure you want to delete all users?"
msgstr ""

#: daemon/rwi.py:1623
#, python-format
msgid "Delete user %s?"
msgstr ""

#: daemon/rwi.py:1651
msgid "Unable to remove user(s) {0} from group {1} (was: {2})."
msgstr ""

#: daemon/rwi.py:1692
msgid "You must specify a file name"
msgstr ""

#: daemon/rwi.py:1722
msgid "Deleting groups ({0}/{1}), progression: {3:.2f}%"
msgstr ""

#: daemon/rwi.py:1752
msgid "Are you sure you want to delete all groups?"
msgstr ""

#: daemon/rwi.py:1770
#, python-format
msgid "Delete group %s?"
msgstr ""

#: daemon/rwi.py:1857
msgid "Are you sure you want to delete all profiles?"
msgstr ""

#: daemon/rwi.py:1873
#, python-format
msgid "Delete profile %s?"
msgstr ""

#: daemon/rwi.py:1915
msgid "Are you sure you want to delete all privileges?"
msgstr ""

#: daemon/rwi.py:1929
#, python-format
msgid "Delete privilege %s?"
msgstr ""

#: daemon/rwi.py:1962
msgid "Are you sure you want to modify all users?"
msgstr ""

#: daemon/rwi.py:1983
#, python-format
msgid "Modify user %s?"
msgstr ""

#: daemon/rwi.py:2012
#, python-format
msgid "Please enter new password for user %s: "
msgstr ""

#: daemon/rwi.py:2014
#, python-format
msgid "Please confirm new password for user %s: "
msgstr ""

#: daemon/rwi.py:2019
msgid "Please enter your OLD (current) password: "
msgstr ""

#: daemon/rwi.py:2021
msgid "Please enter your new password: "
msgstr ""

#: daemon/rwi.py:2022
msgid "Please confirm your new password: "
msgstr ""

#: daemon/rwi.py:2025
#, python-format
msgid "Please enter %s's OLD (current) password: "
msgstr ""

#: daemon/rwi.py:2032
msgid "Wrong password, procedure aborted."
msgstr ""

#: daemon/rwi.py:2042
msgid "Passwords do not match, leaving the old one in place."
msgstr ""

#: daemon/rwi.py:2068 daemon/rwi.py:2076
msgid "Unable to remove user {0} from group {1} (was: {2})."
msgstr ""

#: daemon/rwi.py:2091 daemon/rwi.py:2098
msgid "Unable to add user {0} in group {1} (was: {2})."
msgstr ""

#: daemon/rwi.py:2112
#, python-format
msgid "What do you want to modify about user(s) %s ?"
msgstr ""

#: daemon/rwi.py:2228
msgid "Are you sure you want to modify all groups?"
msgstr ""

#: daemon/rwi.py:2253
#, python-format
msgid "Modify group %s?"
msgstr ""

#: daemon/rwi.py:2259
#, python-format
msgid ""
"Skipped associated system group %s, handled automatically by standard group "
"move."
msgstr ""

#: daemon/rwi.py:2309
msgid "Skipped responsible(s) {0} addition on non-standard group {1}."
msgstr ""

#: daemon/rwi.py:2320
msgid "Skipped responsible(s) {0} deletion on non-standard group {1}."
msgstr ""

#: daemon/rwi.py:2331
msgid "Skipped guest(s) {0} addition on non-standard group {1}."
msgstr ""

#: daemon/rwi.py:2342
msgid "Skipped guest(s) {0} deletion on non-standard group {1}."
msgstr ""

#: daemon/rwi.py:2357
#, python-format
msgid "What do you want to modify about group(s) %s?"
msgstr ""

#: daemon/rwi.py:2380
msgid "Are you sure you want to modify all profiles?"
msgstr ""

#: daemon/rwi.py:2402
#, python-format
msgid "Modify profile %s?"
msgstr ""

#: daemon/rwi.py:2478
#, python-format
msgid "What do you want to modify about profile(s) %s?"
msgstr ""

#: daemon/rwi.py:2530 daemon/rwi.py:2549
msgid "not yet anymore."
msgstr ""

#: daemon/rwi.py:2605 daemon/rwi.py:2614
#, python-format
msgid "Skipped non-existing backend %s."
msgstr ""

#: daemon/rwi.py:2623 daemon/rwi.py:2632
#, python-format
msgid "Skipped non-existing extension %s."
msgstr ""

#: daemon/rwi.py:2675
msgid "What do you want to modify on volumes? use --help to know!"
msgstr ""

#: daemon/rwi.py:2694
msgid "Are you sure you want to check all users?"
msgstr ""

#: daemon/rwi.py:2716
#, python-format
msgid "Check account %s?"
msgstr ""

#: daemon/rwi.py:2738
msgid "Are you sure you want to check all groups?"
msgstr ""

#: daemon/rwi.py:2761
#, python-format
msgid "Check group %s?"
msgstr ""

#: daemon/rwi.py:2789
msgid "Are you sure you want to check all profiles?"
msgstr ""

#: daemon/rwi.py:2803
#, python-format
msgid "Check profile %s?"
msgstr ""

#: daemon/rwi.py:2806
msgid "Sorry, not yet."
msgstr ""

#: daemon/rwi.py:2833
msgid ""
"Django WMI tests are not meant to be run on a production system. If you "
"really want to, set {0} in {1}, and prepare to see your daemon restart after "
"each run."
msgstr ""

#: daemon/rwi.py:2876
msgid "Running WMI Django tests for app {0}, this may take a whileâ¦"
msgstr ""

#: daemon/rwi.py:2897
msgid "Could not run tests for app {0}."
msgstr ""

#: daemon/rwi.py:2900
msgid "Successfully ran Django WMI tests for app {0}."
msgstr ""

#: daemon/cmdlistener.py:164
msgid ""
"Problem finding uid initiating connection from {0}:{1}, denying request."
msgstr ""

#: daemon/cmdlistener.py:192
msgid "Denied connection tentative from {0}:{1} (allowed are {2} and {3})."
msgstr ""

#: daemon/cmdlistener.py:209
msgid ""
"Please implement client.server.bounce auth. Accepted for now (localhost, uid "
"{0})."
msgstr ""

#: daemon/cmdlistener.py:218
msgid "Senseless connection tentative from {0}, uid {2}."
msgstr ""

#: daemon/cmdlistener.py:228
msgid "Connection tentative from {0}: unknown local UID {1}!"
msgstr ""

#: daemon/cmdlistener.py:236
msgid "Something went wrong from {0} (was: {1})."
msgstr ""

#: daemon/cmdlistener.py:248
msgid "Remote login {0} is different from local {1} for UID {2}. Using {0}."
msgstr ""

#: daemon/cmdlistener.py:271
msgid "{0}/{1}: authorized client connection from {2}:{3}, user {4}({5})."
msgstr ""

#: daemon/cmdlistener.py:281
msgid "Connection tentative from {0}:{1}, user {2}({3})."
msgstr ""

#: daemon/cmdlistener.py:413
msgid "{0}: not starting because initial check in progress."
msgstr ""

#: daemon/cmdlistener.py:477
msgid "{0}: {1} to answer requests at {2}."
msgstr ""

#: tests/core.py:40
#, python-format
msgid "%s does not exist on this system and is mandatory for this testsuite."
msgstr ""

#: tests/core.py:45
msgid "Please install missing packages before continuing."
msgstr ""

#: tests/core.py:137
msgid "giving back all scenarii data to {0}:{1}."
msgstr ""

#: tests/core.py:158
msgid "Testsuite terminated successfully."
msgstr ""

#: tests/core.py:159
msgid ""
"Don't forget to test massive del/mod/chk with -a argument (not tested "
"because too dangerous)"
msgstr ""

#: tests/core.py:164
msgid "Cleaning testsuite context, please waitâ¦"
msgstr ""

#: tests/__init__.py:86
#, python-format
msgid "Restored initial backend to %s."
msgstr ""

#: tests/__init__.py:119
msgid "Keyboard Interrupt received; cleaning scenario, please waitâ¦"
msgstr ""

#: tests/__init__.py:149
msgid "Started background tests, waiting for any failed scenarioâ¦"
msgstr ""

#: tests/__init__.py:188
msgid "Re-put scenario in the run queue for further execution."
msgstr ""

#: tests/__init__.py:197
msgid "Stopping the run queue worker and cleaning, please waitâ¦"
msgstr ""

#: tests/__init__.py:212
msgid "no more jobs, ending testsuite."
msgstr ""

#: tests/__init__.py:237
#, python-format
msgid "Saved best state #%s (not inclusive)."
msgstr ""

#: tests/__init__.py:301
msgid ""
"TS global status: on #{current}, {ran}/{total} passed ({percent_done:.2}%) "
"{passed_content}- {failed} failed ({percent_failed:.2}%) - best score until "
"now: #{best}"
msgstr ""

#: tests/__init__.py:317
msgid "context setup"
msgstr ""

#: tests/__init__.py:327
msgid "last: {passed}{passed_more} "
msgstr ""

#: tests/__init__.py:330
msgid "â¦"
msgstr ""

#: tests/__init__.py:376
msgid ""
"The {0} testsuite holds {1} scenarii, counting {2} commands (avg of {3} cmds "
"per scenario)."
msgstr ""

#: tests/__init__.py:395 tests/__init__.py:407
msgid "No scenario selected"
msgstr ""

#: tests/__init__.py:404
#, python-format
msgid "Cannot select a higher scenario than the current best_state (%s)"
msgstr ""

#: tests/__init__.py:425
#, python-format
msgid "Scenario number '%s' is not valid."
msgstr ""

#: tests/__init__.py:456
msgid "State file deleted, start from beginning."
msgstr ""

#: tests/__init__.py:536
msgid ""
"no reference output for {sce_name}, cmd #{cmd_num}/{total_cmds}:\n"
"\n"
"{highlight_cmds}\n"
"\n"
"{run_output}"
msgstr ""

#: tests/__init__.py:546
msgid "is this output good to keep as reference for future runs?"
msgstr ""

#: tests/__init__.py:576
msgid ""
"command #{cmd_num}/{total_cmds} failed (sce#{sce_num}, ctx {context}). "
"Retcode {ret_code} (ref {ref_code}).\n"
"\n"
"{highlight_cmds}\n"
"\n"
"{diff_output}"
msgstr ""

#: tests/__init__.py:591
msgid ""
"Should I keep the new return code and trace as reference for future runs?"
msgstr ""

#: tests/__init__.py:607
msgid "Backend changed to shadow"
msgstr ""

#: tests/__init__.py:616
msgid "Backend changed to OpenLDAP"
msgstr ""

#: tests/__init__.py:706
msgid ""
"Exception {exc} while loading output of cmd {cmd}, sce {sce}. Traceback "
"follows, raising DoesntExistException"
msgstr ""

#: tests/__init__.py:741
#, python-format
msgid ""
"Start run %s. Please wait while commands are executed, this can take some "
"time."
msgstr ""

#: tests/__init__.py:777
#, python-format
msgid "Already passed scenario %s"
msgstr ""

#: tests/__init__.py:798
msgid ""
"Automatically saved new output for scenario #{sce} command #{cmd}/{total}"
msgstr ""

#: tests/__init__.py:808
msgid "Checking FAILED cmd {cmd}/{total} of scenario {sce}"
msgstr ""

#: tests/__init__.py:822
#, python-format
msgid "End run %s."
msgstr ""

#: tests/__init__.py:838
msgid "running "
msgstr ""

#: tests/__init__.py:874
msgid "The last command failed to execute, or returned an unexpected result!"
msgstr ""

#: tests/__init__.py:908
#, python-format
msgid "Cleaning directory %s."
msgstr ""

#: tests/__init__.py:912
#, python-format
msgid "Deleting %s."
msgstr ""

#: tests/__init__.py:922
#, python-format
msgid "Cleaned directory %s."
msgstr ""

#: tests/__init__.py:943
#, python-format
msgid "Backed up system files for context %s."
msgstr ""

#: tests/__init__.py:946
msgid ""
"Comparing backups of system files after tests for side-effects alterations."
msgstr ""

#: tests/__init__.py:967
msgid "System config files backup comparison finished successfully."
msgstr ""

#: tests/__init__.py:988
msgid "reload testsuite. Start from beginning."
msgstr ""

#: tests/__init__.py:990
msgid "execute a specific scenario of the testsuite."
msgstr ""

#: tests/__init__.py:992
msgid "List all scenarii of the testsuite."
msgstr ""

#: tests/__init__.py:994
msgid "Select all scenarii."
msgstr ""

#: tests/__init__.py:996
msgid "Start from the scenario N."
msgstr ""

#: tests/__init__.py:998
msgid "Display messages during command execution."
msgstr ""

#: tests/__init__.py:1000
msgid "Clean scenarii directory."
msgstr ""

#: tests/__init__.py:1002
msgid "Delete traces of a given scenario."
msgstr ""

#: tests/__init__.py:1004
msgid "Display statistics of the testsuite."
msgstr ""

#: tests/__init__.py:1007
msgid ""
"Run the testsuite in standard interactive mode (one scenario at a time. Use "
"this when you have modified a mega bunch of code, and you know it will be "
"better to check everything manually before doing anything else, and the "
"batch run is not what you want because your code is still in alpha stage."
msgstr ""

#: tests/__init__.py:1014
msgid ""
"Don't halt the scenario on fail, just accept the result of the failed "
"command and continue. WARNING: this flag is meant to be used only when you "
"don't have any scenario data in your repository, to build a new one from "
"scratch. Use this flag onlyon a clean source tree, else your TS results will "
"not be reliable."
msgstr ""

#: foundations/pyutils.py:51
msgid "Exception {0} while running {1}({2}{3}{4})"
msgstr ""

#: foundations/pyutils.py:124 foundations/pyutils.py:160
msgid "{0:.2f}Yib"
msgstr ""

#: foundations/pyutils.py:128 foundations/pyutils.py:164
msgid "{0:.2f}Zib"
msgstr ""

#: foundations/pyutils.py:132 foundations/pyutils.py:168
msgid "{0:.2f}Eib"
msgstr ""

#: foundations/pyutils.py:136 foundations/pyutils.py:172
msgid "{0:.2f}Pib"
msgstr ""

#: foundations/pyutils.py:140 foundations/pyutils.py:176
msgid "{0:.2f}Tib"
msgstr ""

#: foundations/pyutils.py:144 foundations/pyutils.py:180
msgid "{0:.2f}Gib"
msgstr ""

#: foundations/pyutils.py:148 foundations/pyutils.py:184
msgid "{0:.2f}Mib"
msgstr ""

#: foundations/pyutils.py:152 foundations/pyutils.py:188
msgid "{0:.2f}Kib"
msgstr ""

#: foundations/pyutils.py:205
#, python-format
msgid "%s ago"
msgstr ""

#: foundations/pyutils.py:208
#, python-format
msgid "in %s"
msgstr ""

#: foundations/pyutils.py:217
msgid "{0} year{1}"
msgstr ""

#: foundations/pyutils.py:218
msgid "{0} month{1}"
msgstr ""

#: foundations/pyutils.py:219
msgid "{0} day{1}"
msgstr ""

#: foundations/pyutils.py:220
msgid "{0} hour{1}"
msgstr ""

#: foundations/pyutils.py:221
msgid "{0} min{1}"
msgstr ""

#: foundations/pyutils.py:222
msgid "{0} sec{1}"
msgstr ""

#: foundations/pyutils.py:223
msgid "{0:.4f} sec{1}"
msgstr ""

#: foundations/pyutils.py:228
msgid "{0}y{1}"
msgstr ""

#: foundations/pyutils.py:229
msgid "{0}m{1}"
msgstr ""

#: foundations/pyutils.py:230
msgid "{0}d{1}"
msgstr ""

#: foundations/pyutils.py:318
msgid "Inserted missing directive {1} in {0}."
msgstr ""

#: foundations/pyutils.py:329
msgid "modified {0} directive {1} to be equal to {2} (was originaly {3})."
msgstr ""

#: foundations/pyutils.py:345
msgid "Modify {0} on disk to reflect current in-memory changes?"
msgstr ""

#: foundations/pyutils.py:351
msgid "Altered {0} to match {1} pre-requisites."
msgstr ""

#: foundations/pyutils.py:357
msgid ""
"Insufficient permissions. Are you root?\n"
"\t{0}"
msgstr ""

#: foundations/pyutils.py:362
msgid ""
"Modifications in {0} are mandatory for {1} to work properly. Cannot continue "
"without this, sorry!"
msgstr ""

#: foundations/workers.py:62
msgid ""
"Cannot join the serviceQ from a ServiceWorkerThread instance, this would "
"deadblock!"
msgstr ""

#: foundations/workers.py:79
msgid ""
"Cannot join the networkQ from a NetworkWorkerThread instance, this would "
"deadblock!"
msgstr ""

#: foundations/workers.py:96
msgid ""
"Cannot join the ackcheckerQ from a ACLCkeckerThread instance, this would "
"deadblock!"
msgstr ""

#: foundations/process.py:58
msgid "{0}({1}): fork #1."
msgstr ""

#: foundations/process.py:68
msgid "{0}({1}): exit parent #1."
msgstr ""

#: foundations/process.py:73
msgid "{0}({1}): fork #1 failed: errno {2} ({3})."
msgstr ""

#: foundations/process.py:79
msgid "{0}({1}): fork #2."
msgstr ""

#: foundations/process.py:85
msgid "{0}({1}): exit parent #2."
msgstr ""

#: foundations/process.py:90
msgid "{0}({1}): fork #2 failed: errno {2} ({3})."
msgstr ""

#: foundations/process.py:94
msgid "{0}({1}): closing all FDs."
msgstr ""

#: foundations/process.py:100
msgid "{0}({1}): ignoring TTY-related signals."
msgstr ""

#: foundations/process.py:128
msgid "{0}({1}): using {2} as log channel."
msgstr ""

#: foundations/process.py:171
#, python-format
msgid "Cannot set process name (was %s)."
msgstr ""

#: foundations/process.py:206
#, python-format
msgid ""
"group %s does not exist and we are not root, aborting. Please manually "
"relaunch this program with root privileges to automatically create this "
"group."
msgstr ""

#: foundations/process.py:220
#, python-format
msgid "Re-exec() ourselves with sudo to gain root privileges (execvp(%s))."
msgstr ""

#: foundations/process.py:226
#, python-format
msgid "You are not a member of group %s; cannot do anything for you, sorry!"
msgstr ""

#: foundations/process.py:232
msgid "Forking licornd."
msgstr ""

#: foundations/process.py:254
msgid "licornd fork failed: errno {0} ({1})."
msgstr ""

#: foundations/process.py:296
msgid "{0:s}: dry_run({1}){2}."
msgstr ""

#: foundations/process.py:299
msgid " â sleep({0})"
msgstr ""

#: foundations/process.py:316
msgid "{0}: Exception while trying to run {1}."
msgstr ""

#: foundations/process.py:439
msgid "Cannot find client 127.0.0.1:{0} or 127.0.1.1:{0} in /proc/net/tcp!"
msgstr ""

#: foundations/process.py:467
msgid "Skipped process @{0} which is not in the same cgroup."
msgstr ""

#: foundations/process.py:515
msgid "Thread {0}{1}: {2}, but no more info."
msgstr ""

#: foundations/process.py:517
msgid "alive"
msgstr ""

#: foundations/process.py:517
msgid "dead"
msgstr ""

#: foundations/network.py:83
msgid ""
"Using environment variable value {0} for our server; unset {1} if you prefer "
"automatic detection."
msgstr ""

#: foundations/network.py:102
msgid "Cannot resolve host or IP {0}"
msgstr ""

#: foundations/network.py:106
msgid "find_server() not implemented yet for your distro!"
msgstr ""

#: foundations/logging.py:104
msgid "Killing ourselves with KILL signal!"
msgstr ""

#: foundations/logging.py:299
msgid ""
"Thread {0} has no listener anymore, desengaging from monitors (was: {1})."
msgstr ""

#: foundations/logging.py:308
msgid ""
"Thread {0} has lost its remote end, desengaging from monitors (was: {1})."
msgstr ""

#: foundations/messaging.py:38
msgid "{0}.remote_output() aborted, no listener registered"
msgstr ""

#: foundations/messaging.py:178
msgid "New trace session started with UUID {0}, facilities {1}."
msgstr ""

#: foundations/messaging.py:218
#, python-format
msgid "Monitor listener with UUID %s not found!"
msgstr ""

#: foundations/messaging.py:220
msgid "Trace session UUID {0} ended."
msgstr ""

#: foundations/messaging.py:266
msgid "{0}: Interactive console requested by {1} from {2}."
msgstr ""

#: foundations/messaging.py:273
msgid "Welcome into licornd's arcanesâ¦"
msgstr ""

#: foundations/messaging.py:276
msgid ">>> Entered batched remote console."
msgstr ""

#: foundations/messaging.py:284
msgid "{0}: Interactive console terminated by {1} from {2}."
msgstr ""

#: foundations/messaging.py:295
msgid "Welcome back to RealÂ Worldâ¢. HaveÂ aÂ niceÂ day!"
msgstr ""

#: foundations/messaging.py:298
msgid ">>> batched remote console terminated."
msgstr ""

#: foundations/_settings.py:82
msgid ""
"Experimental features enabled. Have fun, and hope it does not break anything."
msgstr ""

#: foundations/_settings.py:138
msgid "{0}: internal debug enabled for {1}"
msgstr ""

#: foundations/_settings.py:154
msgid ""
"{0} is currently unset or invalid in {1}. Please set it to either {2} or {3} "
"and retry."
msgstr ""

#: foundations/_settings.py:169
msgid "Trying to find our LicornÂ® serverâ¦"
msgstr ""

#: foundations/_settings.py:172
msgid ""
"Could not find our LicornÂ® server via autodetection. Please contact your "
"network administrator or set the {0} environment variable."
msgstr ""

#: foundations/_settings.py:176
msgid "Our LicornÂ® server is {0}"
msgstr ""

#: foundations/_settings.py:257
msgid "Exception while loading {0}; no exclusions defined"
msgstr ""

#: foundations/_settings.py:279
msgid "Unable to remove {0} from inotifier exclusions"
msgstr ""

#: foundations/_settings.py:294
msgid "Exception while saving {0}!"
msgstr ""

#: foundations/bootstrap.py:188
msgid ""
"You may want to install optional python module{0} {1} to benefit of full "
"functionnality (debian package{2} {3}). As this is optional, you will not "
"see this message again.\n"
msgstr ""

#: foundations/bootstrap.py:194
msgid ""
"You must install required python module{0} {1} (Debian/PIP package{2} {3}) "
"before continuing.\n"
msgstr ""

#: foundations/fsapi.py:308 foundations/fsapi.py:332
#, python-format
msgid "Ignoring unexisting entry \"%s\"."
msgstr ""

#: foundations/fsapi.py:387 foundations/fsapi.py:396
msgid "Impossible to apply this perm or ACL."
msgstr ""

#: foundations/fsapi.py:528 foundations/fsapi.py:540
#, python-format
msgid "posix1e.ACL(text=%s).check() call failed."
msgstr ""

#: foundations/fsapi.py:561
#, python-format
msgid "Do not know how to expand tildes for %s objects!"
msgstr ""

#: foundations/fsapi.py:578
msgid "minifind: mindepth must be <= maxdepth."
msgstr ""

#: foundations/fsapi.py:582
msgid "minifind: please do not try to exhaust maxdepth."
msgstr ""

#: foundations/fsapi.py:622 foundations/fsapi.py:658
msgid "fsapi.minifind(): error on {0}: {1}"
msgstr ""

#: foundations/fsapi.py:641
msgid "minifind(): skipping link or mountpoint {0}."
msgstr ""

#: foundations/fsapi.py:688
#, python-format
msgid "Directory %s does not exist. Create it?"
msgstr ""

#: foundations/fsapi.py:706
msgid "Created directory {0}."
msgstr ""

#: foundations/fsapi.py:710
msgid "Cannot continue checks for directory {0} (was: {1})."
msgstr ""

#: foundations/fsapi.py:722
#, python-format
msgid "Checking file %sâ¦"
msgstr ""

#: foundations/fsapi.py:732
#, python-format
msgid "Checking directory %sâ¦"
msgstr ""

#: foundations/fsapi.py:756
msgid "Checking {1} on {0}â¦"
msgstr ""

#: foundations/fsapi.py:757 foundations/fsapi.py:776
msgid "ACLs"
msgstr ""

#: foundations/fsapi.py:758 foundations/fsapi.py:777
msgid "posix perms"
msgstr ""

#: foundations/fsapi.py:775
msgid "Checking {1} on {0} contentsâ¦"
msgstr ""

#: foundations/fsapi.py:798
#, python-format
msgid "Not touching %s, it is not a file nor a directory."
msgstr ""

#: foundations/fsapi.py:822
msgid "You must pass something through dirs_infos to check!"
msgstr ""

#: foundations/fsapi.py:836
msgid "Invalid utf8 filename \"{0}\". Rename it?"
msgstr ""

#: foundations/fsapi.py:843
msgid " (invalid utf8 filename)"
msgstr ""

#: foundations/fsapi.py:849
msgid " (invalid utf8 filename, copy #{0})"
msgstr ""

#: foundations/fsapi.py:856
msgid "Renamed file {0} to {1}."
msgstr ""

#: foundations/fsapi.py:862
msgid ""
"File {0} has an invalid utf8 filename, it must be renamed before beiing "
"checked further."
msgstr ""

#: foundations/fsapi.py:871
msgid "Do you want to try to continue checking {0} despite of this exception?"
msgstr ""

#: foundations/fsapi.py:902
msgid "{0}: check of symlink {1} skipped."
msgstr ""

#: foundations/fsapi.py:911
msgid ": exception while checking filename validity of {0}."
msgstr ""

#: foundations/fsapi.py:921
msgid ""
"fsapi.check_perms(): Error while checking filename validity of {0}, cannot "
"continue."
msgstr ""

#: foundations/fsapi.py:930
msgid "{0}: checking permissions on {1}â¦"
msgstr ""

#: foundations/fsapi.py:956 foundations/fsapi.py:984
msgid "fsapi.check_perms(): exception while `execbits2str()` on {0}."
msgstr ""

#: foundations/fsapi.py:1018
msgid "Checking {0} of {1}â¦"
msgstr ""

#: foundations/fsapi.py:1019
msgid "posix.1e ACL"
msgstr ""

#: foundations/fsapi.py:1021
msgid "posix permissions"
msgstr ""

#: foundations/fsapi.py:1034
msgid "Exception while getting the posix.1e ACL of {0}"
msgstr ""

#: foundations/fsapi.py:1049
msgid ""
"Invalid access ACL for {path} (it is {current_acl} but should be "
"{access_acl})."
msgstr ""

#: foundations/fsapi.py:1054 foundations/fsapi.py:1131
msgid "empty"
msgstr ""

#: foundations/fsapi.py:1087
msgid "Applyed access ACL {access_acl} on {path}."
msgstr ""

#: foundations/fsapi.py:1096
msgid "fsapi.check_perms(): Exception while setting a posix.1e ACL on {0}"
msgstr ""

#: foundations/fsapi.py:1126
msgid ""
"Invalid default ACL for {path} (it is {current_acl} but should be "
"{access_acl})."
msgstr ""

#: foundations/fsapi.py:1159
msgid "Applyed default ACL {access_acl} on {path}."
msgstr ""

#: foundations/fsapi.py:1169
msgid "fsapi.check_perms(): exception while setting a default ACL on {0}"
msgstr ""

#: foundations/fsapi.py:1189 foundations/fsapi.py:1201
msgid "Exception while looking for an ACL on {0}"
msgstr ""

#: foundations/fsapi.py:1212
msgid "An ACL is present on {path}, but it should not."
msgstr ""

#: foundations/fsapi.py:1230
msgid "Deleted default ACL from {0}."
msgstr ""

#: foundations/fsapi.py:1243
msgid "Deleted access ACL from {0}."
msgstr ""

#: foundations/fsapi.py:1247
msgid "Exception while deleting the posix.1e ACL on {0}"
msgstr ""

#: foundations/fsapi.py:1262 foundations/fsapi.py:1337
msgid "Exception while trying to `stat()` {0}"
msgstr ""

#: foundations/fsapi.py:1273
msgid ""
"Invalid posix permissions for {path} (it is {current_mode} but should be "
"{wanted_mode})."
msgstr ""

#: foundations/fsapi.py:1291
msgid "Applyed posix permissions {wanted_mode} on {path}."
msgstr ""

#: foundations/fsapi.py:1298
msgid "Exception while trying to change posix permissions on {0}."
msgstr ""

#: foundations/fsapi.py:1321
#, python-format
msgid "Checking POSIX uid/gid/perms of %s."
msgstr ""

#: foundations/fsapi.py:1375
msgid ""
"Invalid owership for {0}: currently {1}:{2} but should be {3}:{4}. Correct "
"it?"
msgstr ""

#: foundations/fsapi.py:1399
msgid "Changed owner of {0} to {1}:{2}."
msgstr ""

#: foundations/fsapi.py:1412
msgid "Invalid owership for {0}: currently {1}:{2} but should be {3}:{4}."
msgstr ""

#: foundations/fsapi.py:1428
msgid "Created symlink {link}, pointing to {orig}."
msgstr ""

#: foundations/fsapi.py:1441
msgid ""
"A symlink {link} already exists, but points to {dest}, instead of {good}. "
"Correct it?"
msgstr ""

#: foundations/fsapi.py:1454
msgid "Overwritten symlink {link} with destination {good} instead of {dest}."
msgstr ""

#: foundations/fsapi.py:1463
msgid "Cannot create symlink {0} to {1}!"
msgstr ""

#: foundations/fsapi.py:1469
msgid ""
"Symlink {link} is currently broken, pointing to non-existing target {dest}); "
"making it point to {good}."
msgstr ""

#: foundations/fsapi.py:1485
#, python-format
msgid "Repaired vanished symlink %s."
msgstr ""

#: foundations/fsapi.py:1488
msgid ""
"{link} already exists but it is not a symlink, thus it does not point to "
"{dest}. Rename and replace it with a correct symlink?"
msgstr ""

#: foundations/fsapi.py:1499
msgid "{0} ({1} conflict){2}"
msgstr ""

#: foundations/fsapi.py:1504
msgid ""
"Renamed {link} to {newname} and replaced it by a symlink pointing to {dest}."
msgstr ""

#: foundations/fsapi.py:1513
msgid ""
"While making symlink to {dest}, the destination {link} already exists and is "
"not a link."
msgstr ""

#: foundations/fsapi.py:1531
#, python-format
msgid "Cannot remove %s, it does not exist!"
msgstr ""

#: foundations/fsapi.py:1555
msgid "Archived {0} as {1}."
msgstr ""

#: foundations/fsapi.py:1562
#, python-format
msgid "Cannot archive %s, it does not exist!"
msgstr ""

#: foundations/fsapi.py:1682
msgid "Cloning ACLs is not yet implemented in fsapi.clone_stat(), sorry."
msgstr ""

#: foundations/fsapi.py:1700
msgid "Backed up {orig} as {backup}."
msgstr ""

#: foundations/fsapi.py:1919
msgid "Remounted {0} to apply new mount options."
msgstr ""

#: foundations/fsapi.py:1969
msgid "Added option(s) {0} to {1} permanently."
msgstr ""

#: foundations/base.py:49
msgid ""
"Unbound method {0}() must be called with {1} instance as first argument (got "
"nothing instead)"
msgstr ""

#: foundations/ltrace.py:81
msgid ""
"\n"
"# Thread: {0}({1})"
msgstr ""

#: foundations/ltrace.py:83
msgid "\tFile: \"{0}\", line {1}, in {2}"
msgstr ""

#: foundations/ltrace.py:135
msgid "stack frame unavailable in runtime-compiled code."
msgstr ""

#: foundations/ltrace.py:332
msgid "LicornÂ®: {0} for {1}\n"
msgstr ""

#: foundations/hlstr.py:241 foundations/hlstr.py:260
msgid ""
"Ambiguous word {0}, matches at least {1} and {2}. Please refine your query."
msgstr ""

#: foundations/ttyutils.py:30
msgid " [Ynas], or ? for help: "
msgstr ""

#: foundations/ttyutils.py:87 foundations/ttyutils.py:140
msgid "Yes"
msgstr ""

#: foundations/ttyutils.py:91 foundations/ttyutils.py:143
msgid "No"
msgstr ""

#: foundations/ttyutils.py:139 foundations/ttyutils.py:164
msgid "y"
msgstr ""

#: foundations/ttyutils.py:142 foundations/ttyutils.py:165
msgid "n"
msgstr ""

#: foundations/ttyutils.py:145 foundations/ttyutils.py:166
msgid "a"
msgstr ""

#: foundations/ttyutils.py:147
msgid "Yes, all"
msgstr ""

#: foundations/ttyutils.py:150 foundations/ttyutils.py:167
msgid "s"
msgstr ""

#: foundations/ttyutils.py:152
msgid "No and skip all"
msgstr ""

#: foundations/ttyutils.py:155
msgid "h"
msgstr ""

#: foundations/ttyutils.py:156
msgid ""
"\n"
"\n"
"Usage:\n"
"{0}: fix the current problem and continue until next.\n"
"{1}: do not fix the current problem, skip to next (if possible).\n"
"{2}: fix this problem and all remaining without asking further questions.\n"
"{3}: skip this problem and all remaining ones (if possible)."
msgstr ""

#: foundations/ttyutils.py:171
msgid "wrong command piped on stdin!"
msgstr ""

#: foundations/ttyutils.py:299
#, python-format
msgid "%s: killing ourselves badly."
msgstr ""

#: foundations/ttyutils.py:309
msgid ""
"\n"
"\tInteractive session commands:\n"
"\t\"{0}\", \"{1}\" or \"{2}\": display this help message,\n"
"\t{3}: display a new-line (for readability purposes),\n"
"\t{4}: clear the screen (idem),\n"
"\t{5}: interrupt the current program (kill -INT self),\n"
"\t{6}: terminate (kill -TERM self),\n"
"\t{7}: die badly (kill -KILL self),\n"
msgstr ""

#: foundations/ttyutils.py:319
msgid "Return"
msgstr ""

#: foundations/ttyutils.py:325
msgid "\"{letter_lower}\" or \"{letter_upper}\": "
msgstr ""

#: foundations/ttyutils.py:329
msgid "\" \": "
msgstr ""

#: foundations/ttyutils.py:337
msgid "{0}: received unhandled char \"{1}\", ignoring."
msgstr ""

#: foundations/ttyutils.py:342
msgid "{0}: bad character received on input, ignored."
msgstr ""

#: foundations/events.py:68
msgid "Event push to collector: {0} > {1}"
msgstr ""

#: foundations/events.py:74
msgid "Exception while when pushing event {0} to collector {1}"
msgstr ""

#: foundations/events.py:109
msgid "{0}: no {1} for event {2}."
msgstr ""

#: foundations/events.py:129
msgid ""
"{0}: {1} {2} of event {3} raised a {4} but it has no effect because the "
"event is run asynchronously!"
msgstr ""

#: foundations/events.py:138
msgid ""
"{0}: exception encountered while running {1} {2} for event {3}, continuing "
"with next method"
msgstr ""

#: foundations/events.py:162
msgid "Processing event {0}"
msgstr ""

#: foundations/events.py:165 foundations/events.py:309
#: foundations/events.py:311
msgid "event handler"
msgstr ""

#: foundations/events.py:166 foundations/events.py:305
#: foundations/events.py:307
msgid "event callback"
msgstr ""

#: foundations/events.py:195
#, python-format
msgid "A synchronous event cannot be delayed! (on %s)"
msgstr ""

#: foundations/events.py:330
msgid "{0}: {1} {2} already registered for event {3}."
msgstr ""

#: foundations/events.py:343
msgid "{0}: registered {1} as new {2} for event {3}."
msgstr ""

#: foundations/events.py:356
msgid "{0}: unregistered {2} {1} for event {3}."
msgstr ""

#: foundations/events.py:362
msgid "{0}: event \"{1}\" not found when trying to unregister {2} {3}."
msgstr ""

#: foundations/events.py:369
msgid "{0}: {1} {2} already not registered for event {3}."
msgstr ""

#: foundations/events.py:388
msgid "{0}: registered event collector {1}."
msgstr ""

#: foundations/events.py:404
msgid "{0}: unregistered event collector {0}."
msgstr ""

#: foundations/events.py:410
msgid "Error while trying to unregister collector {0}"
msgstr ""

#: foundations/events.py:475
msgid "{0}: LicornÂ® Event Loop started."
msgstr ""

#: foundations/events.py:488
msgid "{0}: LicornÂ® Event Loop stopped."
msgstr ""

#: foundations/events.py:501
msgid ""
"{0}{1}: {2} events, {3} handler(s) and {4} callback(s) registered,\n"
"{5}\n"
"{6}"
msgstr ""

#: foundations/events.py:520
msgid "{0}{1} ({2} events, {3} handler(s) and {4} callback(s) registered)"
msgstr ""

#: foundations/apt.py:21
msgid ""
"You must install the debian package {0} for the upgrade mechanism to work "
"fully. Currently you will only be able to check if the system needs "
"upgrading."
msgstr ""

#: foundations/apt.py:56
msgid "Error while running Â«Â unattended-upgradesÂ Â»!"
msgstr ""

#: foundations/cache.py:61
#, python-format
msgid "Key %s expired!"
msgstr ""

#: foundations/classes.py:104
msgid ""
"{0}: system file {1} must be modified for the configuration to be complete. "
"Do it?"
msgstr ""

#: foundations/classes.py:113
msgid "{0}: altered configuration file {1}."
msgstr ""

#: foundations/classes.py:169
msgid ""
"{0}: File {1} is protected by the {2} flag, we cannot alter it. Hope this is "
"intended, because we are continuing like if it was modified but it is NOT."
msgstr ""

#: foundations/classes.py:173
msgid "IMPORTANT"
msgstr ""

#: foundations/packaging.py:58
msgid ""
"Your distro is not yet supported to automatically install package {0}, "
"skipping it."
msgstr ""

#: foundations/packaging.py:78
msgid "{0}: Installing needed packages {1} before continuing. Please waitâ¦"
msgstr ""

#: foundations/packaging.py:84
msgid ""
"{0}: An error occured while installing package(s) {1}! Apt-Get Log follows:"
msgstr ""

#: foundations/packaging.py:88 foundations/packaging.py:122
msgid "{0}: Successfully installed package(s) {1} via {2}."
msgstr ""

#: foundations/packaging.py:93
msgid ""
"Installing packages on your distro is not yet supported, sorry. You should "
"install {0} yourself before continuing."
msgstr ""

#: foundations/packaging.py:112
msgid ""
"{0}: Installing needed packages {1} from source before continuing. Please "
"waitâ¦"
msgstr ""

#: foundations/packaging.py:118
msgid ""
"An error occured while installing package(s) {0}! PIP install Log follows:"
msgstr ""

#: foundations/packaging.py:125
msgid ""
"Sorry, automatic install of {0} is not yet supported on your distro. Please "
"install it manually before continuing."
msgstr ""

#: foundations/exceptions.py:216
msgid "The file {0} is corrupt!"
msgstr ""

#: foundations/exceptions.py:219
msgid "Reason: {0}"
msgstr ""
